---
csl: mbio.csl
fontsize: 12pt
output:
  word_document: default
  pdf_document:
    includes:
      in_header: header.tex
    keep_tex: yes
bibliography: references.bib
---


```{r knitr_settings, eval=TRUE, echo=FALSE, cache=FALSE, warning=FALSE, message=FALSE}
library(knitr)
library(readr)

opts_chunk$set("tidy" = TRUE)
opts_chunk$set("echo" = FALSE)
opts_chunk$set("eval" = TRUE)
opts_chunk$set("warning" = FALSE)
opts_chunk$set("cache" = FALSE)
opts_chunk$set("message" = FALSE)

```

\pagenumbering{arabic}
\linenumbers
\doublespacing

# The gut bacterial community potentiates *Clostridioides difficile* infection severity.
  


\vspace{30mm}


**Running title:** Microbiota potentiates *Clostridioides difficile* infection severity

\vspace{20mm}

Nicholas A. Lesniak$^1$, Alyxandria M. Schubert$^1$, Kaitlyn Flynn$^1$, Jhansi L Leslie$^1$, Hamide Sinani$^1$, Ingrid L. Bergin$^3$, Vincent B Young$^2$, Patrick D. Schloss$^{1,\dagger}$

\vspace{30mm}

$\dagger$ To whom correspondence should be addressed: pschloss@umich.edu  
1\. Department of Microbiology and Immunology, University of Michigan, Ann Arbor, MI  
2\. Division of Infectious Diseases, Department of Internal Medicine, University of Michigan Medical School, Ann Arbor, MI  
3\. Unit for Laboratory Animal Medicine, University of Michigan, Ann Arbor, MI  




\newpage 


## Abstract

The severity of *Clostridioides difficile* infections (CDI) has increased over the last few decades. Patient age, white blood cell count, creatinine levels as well as *C. difficile* ribotype and toxin genes have been associated with disease severity. However, it is unclear whether there is an associate between the gut microbiota disease severity. The gut microbiota is known to interact with *C. difficile* during infection. Perturbations to the gut microbiota are necessary for *C. difficile* to colonize the gut. The gut microbiota is thought to impair *C. difficile* colonization through bile acid metabolism, nutrient consumption and bacteriocin production. It has not been shown whether the gut microbiota can affect CDI disease severity. Here we demonstrate that gut bacterial communities can contribute to disease severity. We derived diverse gut communities by colonizing germ-free mice with different human fecal communities. The mice were then infected with a single *C. difficile* ribotype 027 clinical isolate which resulted in moribundity and histopathologic differences. The variation in severity was associated with the human fecal community that the mice received. Generally, facultative anaerobes with pathogenic potential, such as *Escherichia*, *Helicobacter*,  and *Klebsiella*, were associated with more severe outcomes. Bacterial groups associated with fiber degradation, such as *Coprobacillus*, were associated with less severe outcomes. Finally, we showed that, when infected with a different *C. difficile* isolate, the outcome of the infection was similar between the mice that received the same fecal donor community. These data indicate that gut bacteria can influence CDI disease severity.

## Importance

*Clostridioides difficile* infections (CDI) can be asymptomatic or range in severity from mild diarrhea to severe outcomes including surgical interventions and death. Models that predict severity and guide treatment decisions are based on clinical factors and *C. difficile* characteristics. Although the gut microbiome plays a role in protecting against CDI, it has not been investigated for its effect on CDI disease severity or incorporated into attempts to predict disease severity. We demonstrated that variation in the microbiome of mice colonized with human feces yielded a range of disease outcomes. These results revealed groups of bacteria associated with both severe and mild *C. difficile* infection outcomes. Gut bacterial community data from patients with CDI could improve our ability to identify patients at risk of developing more severe disease and improve interventions which treat the gut bacteria to reduce host damage.



\newpage


## Introduction

*Clostridioides difficile* infections (CDI) have increased in incidence and severity since *C. difficile* was first identified as the cause of antibiotic-associated pseudomembranous colitis [@Kelly2008]. CDI disease severity can range from asymptomatic to mild diarrhea to toxic megacolon and death. The Infectious Diseases Society of America (IDSA) and Society for Healthcare Epidemiology of America (SHEA) guidelines categorize severe disease based on white blood cell count greater than 15,000 cells/mm$^3$ and/or a serum creatinine greater than 1.5 mg/dL, and complicated disease based on systolic blood pressure below 90, ileus, or toxic megacolon [@McDonald2018]. Since these measures are only present once the disease has advanced, the IDSA classification does not allow for identification of patients at risk of severe CDI. Prediction models have been developed to score a patient's risk for severe CDI outcomes based on clinical factors but have not been robust for broad application [@Dieterle2020; @Butt2013; @Perry2021; @vanBeurden2017]. Thus, we have limited ability to prevent patients from developing severe CDI. Ideally, we could identify patients that are high-risk of severe disease so their treatments could be adjusted to halt the infection before the disease worsens.

Missing from CDI severity prediction models are the effects of the indigenous gut bacteria. *C. difficile* interacts with the gut community in many ways. The indigenous bacteria of a healthy intestinal community provide a protective barrier preventing *C. difficile* from infecting the gut. A range of mechanisms can disrupt this barrier, including antibiotics, medications, or dietary changes, and lead to increased susceptibility to CDI [@Britton2014; @Hryckowian2018; @VichVila2020]. Once in the gut, the indigenous bacteria can either promote or inhibit *C. difficile* through producing molecules or modifying the environment [@Abbas2020; @Jenior2017]. Bile acids metabolized by the gut bacteria can inhibit *C. difficile* growth and affect toxin production [@Sorg2008; @Thanissery2017]. Bacteria in the gut also can compete more directly with *C. difficile* through antibiotic production or nutrient consumption [@Aguirre2021; @Kang2019]. While the relationship between the gut bacteria and *C. difficile* has been established, the effect the gut bacteria can have on CDI disease severity is unclear. 

Studies investigating the severity of CDI disease involving the microbiome have not shown clear evidence of a relationship between the microbiome and disease severity. These studies were not setup to identify a relationship between the microbiome and disease severity. Studies that have used clinical data cannot control for host, microbiome or *C. difficile* ribotype variation [@Seekatz2016]. While disease severity varies, it cannot be attributed to specifically the microbiome since all factors vary. Murine experiments typically use a single mouse colony and different *C. difficile* ribotypes to create severity differences [@Dieterle2020]. Recently, our group has begun uncovering the effect microbiome variation has on *C. difficile* infection. We showed the variation in the bacterial communities between mice from different mouse colonies resulted in different clearance rates of *C. difficile* [@Tomkovich2020]. We also showed varied ability of mice to spontaneously eliminate *C. difficile* infection when they were treated with different antibiotics prior to *C. difficile* challenge [@Lesniak2021]. However, there have not been any studies investigating whether variation in the microbiome affects the resultant disease severity of *C. difficile* infections. 

Here, we sought to determine whether the gut bacteria contribute to disease severity. We hypothesized that since the gut bacteria affect the metabolism of *C. difficile* and its infection dynamics, it can also affect the disease severity of the infection. To test this hypothesis, we colonized germ-free C57BL/6 mice with human fecal communities to create varied gut communities. We then challenged the mice with a single *C. difficile* isolate, ribotype 027 (RT027) human isolate 431, and followed the mice for the development of severe outcomes of moribundity and histopathologic cecal tissue damage. Since the murine host and *C. difficile* isolate were the same and only the gut community varied, the variation in disease severity we observed was attributable to the microbiome. 

## Results

***C. difficile* is able to infect germ-free mice colonized with human fecal microbial communities without perturbation.**  To produce gut microbiomes with greater variation than those found in conventional mouse colonies, we colonized germ-free mice with bacteria from human feces [@Nagpal2018]. We inoculated germ-free C57BL/6 mice with homogenized feces from one of 15 human fecal samples via oral gavage. The human fecal samples were selected because they represented diverse community structures based on community clustering [@Schubert2014]. The gut communities were allowed to establish themselves for two weeks post inoculation [@Gillilland2012]. We then surveyed the bacterial members of the gut communities by 16S rRNA gene sequencing of fecal pellets (Figure 1A). The bacterial communities from each mouse grouped more closely to those communities from mice that received the same human fecal donor community than to the mice who received a different human fecal donor community (Figure 1B). The communities were primarily composed of populations of *Clostridia*, *Bacteroidia*, *Erysipelotrichia*, *Bacilli*, and *Gammaproteobacteria*. However, the gut bacterial communities of each donor group of mice harbored unique relative abundance distributions of the shared bacterial classes.

Next, we tested this set of mice with their human-derived gut microbial communities for susceptibility to *C. difficile* infection. A typical mouse model of CDI requires pre-treatment with antibiotics, such as clindamycin, to become susceptible to *C. difficile* colonization [@Chen2008; @Schubert2015]. However, we wanted to avoid modifying the communities with an antibiotic to maintain their unique microbial compositions and ecological relationships. Since these communities came from people who had increased risk of CDI, such as hospitalization or recent antibiotic use [@Schubert2014], we decided to test if *C. difficile* was able to infect these mice without any antibiotic perturbation. We hypothesized that *C. difficile* would be able to colonize a subset of mice whose communities were susceptible from perturbations either the donor experienced or from the ecological selection when colonizing the murine gut. Mice were challenged with 10$^{3}$ *C. difficile* RT027 human isolate 431 spores. The mice were followed for 10 days post-challenge and their stool was collected and plated for *C. difficile* colony forming units (CFU) to determine the extent of the infection. Surprisingly, mice from all donors were colonized (Figure 2). Only two mice were able to resist *C. difficile* infection and both received their community from the same donor, N3. By colonizing germ-free mice with different human fecal communities, we were able to generate diverse gut communities in mice, which were susceptible to *C. difficile* infection without further modification of the gut community.

**Infection severity varies by initial community.** After we challenged the mice with *C. difficile*, we investigated the outcome from the infection and its relationship to the initial community. We followed the mice for 10 days post-challenge for colonization density, toxin production, and mortality. Seven mice, from Donors N1, N2 and N5, were not colonized on the day after *C. difficile* challenge but were infected (>10$^{6}$) by the end of the experiment. All mice that received their community from Donor M1 through M6 succumbed to the infection and became moribund within 3 days post-challenge. The remaining mice, except the uninfected Donor N3 mice, maintained *C. difficile* infection through the end of the experiment (Figure 2). At 10 days post-challenge, or earlier for the moribund mice, mice were euthanised and fecal pellets were assayed for toxin activity and cecal tissue was collected and scored for histopathologic signs of disease (Figure 3). Overall, there was greater toxin activity detected in the stool of the moribund mice (*P* = 0.003). However, when looking at each group of mice, we observed a range in toxin activity for both the moribund and non-moribund mice (Figure 3A). Non-moribund mice from Donors N4 through N9 had comparable toxin activity as the moribund mice. Additionally, not all moribund mice had toxin activity detected in their stool. This discordance between toxin activity and disease severity has been previously described [@Akerlund2006; @Vitucci2020]. Next we examined the cecal tissue for histopathologic damage. Moribund mice had high levels of epithelial damage, tissue edema, and inflammation (Figure S1) similar to previously reported histopathologic findings for *C. difficile* RT027 [@Cowardin2016]. As observed with toxin activity, the moribund mice had higher histopathologic scores than the non-moribund mice (*P* < 0.001). However, unlike the toxin activity, all moribund mice had consistently high histopathologic summary scores (Figure 3B). The non-moribund mice, Donor groups N1 through N9, had a range in tissue damage from none detected to similar levels as the moribund mice, which grouped by community donor. Together, the toxin activity, histopathologic score, and moribundity showed variation across the donor groups but were largely consistent within each donor group.

**Microbial community members explain variation in CDI severity.** We next interrogated the bacterial communities at the time of *C. difficile* challenge (day 0) for their relationship to infection outcomes using linear discriminant analysis (LDA) effect size (LEfSe) analysis to identify individual bacterial populations that could explain the variation in disease severity. We split the mice into groups by severity level based on their moribundity and histopathologic score. We dichotomized the histopathologic scores into high and low groups by splitting on the median score. This analysis revealed 20 genera that were significantly different by the disease severity (Figure 4A). Bacterial genera *Turicibacter*, *Streptococcus*, *Staphylococcus*, *Pseudomonas*, *Phocaeicola*, *Parabacteroides*, *Bacteroides*, and *Escherichia/Shigella* were detected at higher relative abundances in the mice that became moribund. Populations of *Anaerotignum*, *Coprobacillus*, *Enterocloster*, and *Murimonas* were more abundant in the non-moribund mice that would develop only low intestinal injury. To understand the role of toxin activity in disease severity, we applied LEfSe to identify the genera most likely to explain the observed differences in toxin activity (Figure 4B). Many genera that associated with toxin production were also associated with moribundity, such as populations of *Escherichia/Shigella* and *Bacteroides*. Likewise, there were genera such as *Anaerotignum*, *Enterocloster*, and *Murimonas* that were associated with no toxin production that also exhibited greater relative abundance in communities from non-moribund mice with a low histopathologic score. Lastly, we tested for correlations between the endpoint relative abundances of bacterial operational taxonomic units (OTUs) and the histopathologic summary score (Figure 4C). The endpoint relative abundance of *Bacteroides* was positively correlated with histopathologic score, as its day 0 relative abundance did with disease severity (Figure 4A). Populations of *Klebsiella* and *Prevotellaceae* were positively correlated with the histopathologic score and were increased in the group of mice with detectable toxin. This analysis identified bacterial genera that were associated with the variation in moribundity, histopathologic score, and toxin activity of CDI.

We next determined whether collectively, bacterial community membership and relative abundance could be predictive of the CDI disease outcome. We trained random forest models with bacterial community relative abundance data from the day of colonization at each taxonomic rank to predict toxin production, moribundity, and day 10 post-challenge histopathologic summary score. For predicting if toxin would be produced, microbial populations aggregated by phylum rank classification performed similarly as models using lower taxonomic ranks (AUROC = 0.83, Figure S2). *C. difficile* was more likely to produce toxin when the community it infected had less abundant populations of *Verrucomicrobia* and *Campilobacterota* and had more abundant populations of *Proteobacteria* (Figure 5A). Next, we assessed the ability of the community to predict moribundity. Bacteria grouped by class rank classification was sufficient to predict which mice would succumb to the infection before the end of the experiment (AUROC = 0.91, Figure S2). The features with the greatest effect showed that communities with greater populations of bacteria belonging to *Bacilli* and *Firmicutes* and reduced populations of *Erysipelotrichia* were more likely to result in moribundity (Figure 5B). Only one other class of bacteria was decreased in moribund mice, a group of unclassified *Clostridia*. Lastly, the relative abundances of genera were able to predict a high or low histopathologic score (histopathologic scores were dichotomized as in previous analysis, AUROC = 0.99, Figure S2). No genera had a significantly greater effect on the model performance than any others, indicating the model was reliant on many genera for the correct prediction. The model used some of the genera identified in the LEfSe analysis, such as *Coprobacillus*, *Anaerostipes*, and *Hungatella*. Communities with greater abundances of *Hungatella*, *Eggerthella*, *Bifidobacterium*, *Duncaniella* and *Neisseria* were more likely to have high histopathologic scores. These models have shown that the relative abundance of bacterial populations and their relationship to each other could be used to predict the variation in moribundity, histopathologic score, and toxin activity of CDI.

**Disease severity consistent by donor community across isolates.** Since Carlson et al. showed *C. difficile* isolates with lower germination rates were more likely to result in severe disease [@Carlson2015], we next tested whether the effect of the community on CDI outcomes was conserved across *C. difficile* isolates with higher rates of germination. We colonized mice with human fecal slurry of either Donor N7, N6, or M4 and allowed two weeks for the community to establish. We then challenged the mice with a different *C. difficile* isolate and all mice were infected. The moribundity and histopathologic score outcomes when challenged with a different *C. difficile* human isolate were comparable to the outcomes of challenges with *C. difficile* RT027 human isolate 431 by their donor group (Figure 6). For the isolates and communities we tested, the effect of the gut bacterial community on disease severity was greater than the effect of the variation in these isolates.


## Discussion

Challenging mice colonized with different human fecal communities with a single *C. difficile* isolate, RT027 isolate 431, demonstrated that variation in the gut bacteria affects the *C. difficile* infection disease severity. Our analysis revealed an association between the relative abundance of bacterial community members and disease severity. We used the relative abundance patterns of community members associated with disease outcomes to select donor communities to test whether the disease severity was conserved when challenged with a different *C. difficile* isolate. The disease outcomes from the challenge with the second *C. difficile* isolate matched the outcomes of the challenge with *C. difficile* RT027 isolate 431. Therefore, the effect of the community variation on disease severity *in vivo* was greater than the effect of *C. difficile* isolate variation [@Carlson2013; @Carlson2015]. Overall, these results have demonstrated that the bacterial community contributed to the disease severity of *C. difficile* infection.

CDI severity can range from no symptoms to mild symptoms, such as diarrhea, to severe symptoms, such as toxic megacolon or death. Physicians use prediction tools to identify patients most at risk of developing a severe infection using white blood cell counts, serum albumin level, or serum creatinine level [@Lungulescu2011; @McDonald2018; @Zar2007]. Those levels are driven by the activities in the intestine [@diMasi2018]. Research into the drivers of this variation have revealed factors that make *C. difficile* more virulent. Strains are categorized for their virulence by the presence and production toxin TcdA, toxin TcdB, and binary toxin and the prevalence in outbreaks, such as ribotypes 027 and 078 [@AbernathyClose2020; @Theriot2011; @Goorhuis2008; @OConnor2009; @Chen2008]. However, other studies have shown that virulence is not necessarily linked with toxin production [@Rao2015] or the strain [@Walk2012]. Furthermore, there is variation in the genome, growth rate, sporulation, germination, and toxin production in different isolates of a strain [@Carlson2013; @Carlson2015; @Saund2021; @He2010]. This variation may help explain why severe CDI prediction tools often miss identifying many patients with CDI that will develop severe disease [@Dieterle2020; @Butt2013; @Perry2021; @vanBeurden2017]. Therefore, it is necessary to gain a full understanding of all factors contributing to disease variation to improve our ability to predict severity.

The state of the gut bacterial community determines the ability of *C. difficile* to colonize and persist in the intestine. *C. difficile* is unable to colonize an unperturbed healthy gut community and is only able to become established after a perturbation [@Schubert2015]. Once colonized, the different communities lead to different metabolic responses and dynamics of the *C. difficile* population [@Jenior2017; @Jenior2018; @Lesniak2021]. The gut bacteria metabolize primary bile acids into secondary bile acids [@Staley2016; @Long2017]. The concentration of these bile acids affect the germination, growth, toxin production and biofilm formation [@Sorg2008; @Sorg2010; @Thanissery2017; @Dubois2019]. Members of the bacterial community also affect other metabolites *C. difficile* utilizes. *Bacteroides thetaiotaomicron* produce sialidases which release sialic acid from the mucosa for *C. difficile* to utilize [@Ng2013; @Ferreyra2014]. The nutrient environment affects toxin production [@MartinVerstraete2016]. Thus, many of the actions of the gut bacteria modulate *C. difficile* in ways that could affect the infection and resultant disease.

Myriad studies have explored the relationship between the microbiome and CDI but none have established a relationship between the variation of disease severity and the microbiome. CDI studies often use an infection model with a single homogeneous bacterial community. Collins et al. used multiple human communities to colonize mice, however the communities were pooled prior to gavaging into germ-free mice [@Collins2015], resulting in a single community. Studies examining difference in disease often use different *C. difficile* strains or ribotypes in mice with similar microbiota as a proxy for variation in disease, such as strain 630 for non-severe and RT027 for severe [@AbernathyClose2020; @Theriot2011; @Lawley2012; @Chen2008]. Studies have also demonstrated variation in infection through tapering antibiotic dosage [@Lesniak2021; @Schubert2015; @Reeves2011] or by reducing the amount of *C. difficile* cells or spores used for the challenge [@Chen2008; @Reeves2011]. These studies either lack variation in the initial microbiome or have variation in the *C. difficile* infection itself, therefore we cannot make any association between variation in severity and the microbiome.

With our recent observations that the initial community affected the ability of *C. difficile* to persist in the gut [@Tomkovich2020; @Lesniak2021] and the existing research describing the myriad interactions between the microbiome and *C. difficile*, we hypothesized that the microbiome can modulate CDI disease severity. Our data have demonstrated gut bacterial relative abundances associate with variation in toxin production, histopathologic scoring of the cecal tissue and mortality. This analysis revealed populations of *Akkermansia*, *Anaerostipes*, *Coprobacillus*, *Enterocloster*, *Lactonifactor*, and *Monoglobus* were more abundant in the microbiome of non-moribund mice which had low histopathologic scores and no detected toxin. The protective role of these genera are supported by previous studies. *Coprobacillus*, *Lactonifactor*, and *Monoglobus* have been shown to be involved in dietary fiber fermentation and associated with healthy communities [@Mabrok2011; @Kim2019; @doPrado2021; @Muthuramalingam2019]. *Anaerostipes* and *Coprobacillus*, which produce short chain fatty acids, have been associated with healthy communities [@Han2019; @Duncan2004; @Ye2018]. Furthermore, *Coprobacillus*, which was abundant in mice with low histopathologic scores but rare in all other mice, has been shown to contain a putative type I lantibiotic gene cluster and inhibit *C. difficile* colonization [@Walsh2017; @Sandiford2018; @Stein2013]. *Akkermansia* and *Enterocloster* were also identified as more abundant in mice which had a low histopathologic scores but have contradictory supporting evidence in the current literature. In our data, *Akkermansia* was most abundant in the non-moribund mice with low histopathologic scores but there were some moribund mice which had increased populations of *Akkermansia*. This could be attributed to either a more protective mucus layer was present inhibiting colonization [@Stein2013; @Nakashima2021] or mucus consumption by *Akkermansia* could have been crossfeeding *C. difficile* or exposing a niche for *C. difficile* [@Geerlings2018; @Deng2018; @Engevik2020]. Similarly, *Enterocloster* was more abundant and associated with low histopathologic scores. It has been associated with healthy populations and has been used to mono-colonize germ-free mice to reduce the ability of *C. difficile* to colonize [@Reeves2012; @Ma2021]. However, *Enterocloster* has also been involved in infections, such as bacteremia [@Haas2020; @Finegold2005]. These data have exemplified populations of bacteria that have the potential to be either protective or harmful. Thus, the disease outcome is not likely based on the abundance of individual populations of bacteria, rather it is the result of the interactions of the community.

The groups of bacteria that were associated with either a higher histopathologic score or moribundity are members of the indigenous gut community that also have been associated with disease, often referred to as opportunistic pathogens. Many of the populations with pathogenic potential that associated with worse outcomes are also facultative anaerobes. *Enterococcus*, *Klebsiella*, *Shigella/Escherichia*, *Staphylococcus*, and *Streptococcus* have been shown to expand after antibiotic use [@VanInsberghe2020; @GarzaGonzalez2019; @Schubert2014] and are commonly detected in CDI cases [@Shafiq2020; @Keith2020; @Zackular2016; @Berkell2021]. In addition to these populations, *Eggerthella*, *Prevotellaceae* and *Helicobacter*, which associated with worse outcomes, have also been associated with intestinal inflammation [@Gardiner2014; @Iljazovic2020; @Nagalingam2013]. Recently, *Helicobacter hepaticus* was shown to be sufficient to cause susceptibility to CDI in IL-10 deficient C57BL/6 mice [@AbernathyClose2021]. In our experiments, when *Helicobacter* was present, the infection resulted in a high histopathologic score. While we did not use IL-10 deficient mice, it is possible the bacterial community or host response are similarly modified by *Helicobacter*, allowing *C. difficile* infection and host damage. Aside from *Helicobacter*, these groups of bacteria that associated with more severe outcomes did not have a conserved association between their relative abundance and the disease severity across all mice.

Since we observed groups of bacteria that were associated with less severe disease it may be appropriate to apply the damage-response framework for microbial pathogenesis to CDI [@Pirofski2008; @Casadevall2014]. This framework posits that disease is not driven by a single entity, rather it is an emergent property of the responses of the host immune system, infecting microbe, *C. difficile*, and the indigenous microbes at the site of infection. In the first set of experiments, we used the same host, C57BL/6 mice, the same infecting microbe, *C. difficile* RT027 human isolate 431, with different gut bacterial communities. The bacterial groups in those communities were often present in both moribund and non-moribund and across the range of histopathologic scores. Thus, it was not merely the presence of the bacteria but their activity in response to the other microbes and host which affect the extent of the host damage. Additionally, while each mouse and *C. difficile* population had the same genetic background, they too were reacting to the specific microbial community. Disease severity is driven by the cumulative effect of the host immune response and the activity of  *C. difficile* and the gut bacteria. *C. difficile* can drive host damage through the production of toxin. The gut microbiota can contribute to the host damage through the balance of metabolic and competitive activities, such as bacteriocin production or mucin degradation. For example, low levels of mucin degradation can provide nutrients to other community members producing a diverse non-damaging community [@Tailford2015]. However, if mucin degradation becomes too great it reduces the protective function of the mucin layer and exposes the epithelial cells. This over-harvesting can contribute to the host damage due to other members producing toxin. Thus, the resultant intestinal damage is the balance of all activities in the gut environment. Host damage is the emergent property of numerous damage-response curves, such as one for host immune response, one for *C. difficile* activity and another for microbiome community activity, each of which are a composite curve of the individual activities from each group, such as antibody production, neutrophil infiltration, toxin production, sporulation, fiber and mucin degradation. Therefore, while we have identified populations of interest, it may be necessary to target multiple types of bacteria to reduce the community interactions contributing to host damage.

Here we have shown myriad bacterial groups and their relative abundances associated with variation in severity of CDI disease. Further understanding of how the microbiome affects severity in clinical patients could reduce the amount of adverse outcomes of CDI. When a patient is diagnosed with CDI, the gut community composition, in addition to the traditionally obtained clinical information, may improve our severity prediction and guide prophylactic treatment. Treating the microbiome at the time of diagnosis may prevent the infection from becoming more severe. Based on the general trends of our results, promoting fiber metabolizing bacteria and reducing facultative anaerobes may bolster the efficacy of current CDI treatments, reducing disease severity.


## Materials and Methods

**Animal care.** 6- to 13-week old male and female germ-free C57BL/6 were obtained from a single breeding colony in the University of Michigan Germ-free Mouse Core. Mice were housed in cages of 2-4 mice per cage and maintained in germ-free isolators at the University of Michigan  germ-free facility. All mouse experiments were approved by the University Committee on Use and Care of Animals at the University of Michigan. **Were the mice maintained in gnotobiotic isolators throughout the experiment? Or were they transferred to BSL2 conditions prior to administering the human FMTs?**

***C. difficile* experiments.** Human fecal samples were obtained as part of Schubert et al. and selected based on community clusters [@Schubert2014] to result in diverse community structures. Feces were homogenized by mixing 200 mg of sample with 5 ml of PBS. Mice were inoculated with 100 $\mu$l of the fecal homogenate via oral gavage. For the set of experiments testing multiple *C. difficile* clinical isolates, we selected a subset of donor fecal samples based on their associated mouse transplant communities and the outcomes in the experiments with *C. difficile* RT027 isolate 431 that fit the following criteria: one community we would expect to result in a low histopathologic score (< 5), a high histopathologic score (> 5), and a high histopathologic score which becomes moribund. Additionally, we selected the donors based their relative abundances of genera identified through the our analysis (Figure S3). Two weeks after the fecal community inoculation, mice were challenged with *C. difficile*. *C. difficile* isolates came from Carlson et al. which had previously been isolated and characterized [@Carlson2013; @Carlson2015] and have recently been further characterized [@Saund2021]. Spores concentration were determined both before and after challenge [@Sorg2009]. $10^{3}$ *C. difficile* spores were given to each mouse via oral gavage. **How were human FMTs prepared (anaerobically vs aerobically, etc)?**

**Sample collection.** Fecal samples were collected on the day of *C. difficile* challenge and the following 10 days. Each day, a fecal sample was also collected and weighed and the remaining sample was frozen at -20$^\circ$C. Anaerobically, the weighed fecal samples were serially diluted in PBS, plated on TCCFA plates, and incubated at 37$^\circ$C for 24 hours. The plates were then counted for the number of colony forming units (CFU) [@Winston2016].

**DNA sequencing.** From the frozen fecal samples, total bacterial DNA was extracted using MOBIO PowerSoil-htp 96-well soil DNA isolation kit. We amplified the 16S rRNA gene V4 region and sequenced the resulting amplicons using an Illumina MiSeq as described previously [@Kozich2013].

**Sequence curation.** Sequences were processed with mothur(v.1.44.3) as previously described [@Kozich2013; @Schloss2009]. In short, we used a 3% dissimilarity cutoff to group sequences into operational taxonomic units (OTUs). We used a naive Bayesian classifier with the Ribosomal Database Project training set (version 18) to assign taxonomic classifications to each OTU [@Wang2007]. We sequenced a mock community of a known community composition and 16s rRNA gene sequences. We processed this mock community with our samples to calculate the error rate for our sequence curation, which was an error rate of 0.19%.

**Toxin cytotoxicity assay.** To prepare the sample for the activity assay, fecal material was diluted 1:10 weight per volume using sterile PBS and then filter sterilized through a 0.22-$\mu$m filter. Toxin activity was assessed using a Vero cell rounding-based cytotoxicity assay as described previously [@Theriot2011]. The cytotoxicity titer was determined for each sample as the last dilution, which resulted in at least 80% cell rounding. Toxin titers are reported as the log10 of the reciprocal of the cytotoxicity titer.

**Histopathology evaluation.** Mouse cecal tissue was placed in histopathology cassettes and fixed in 10% formalin, then stored in 70% ethanol. McClinchey Histology Labs, Inc. (Stockbridge, MI) embedded the samples in paraffin, sectioned, and created the hematoxylin and eosin-stained slides. The slides were scored using previously described criteria by a board-certified veterinary pathologist scored who was blinded to the experimental groups [@Theriot2011].

**Statistical analysis and modeling.** To compare community structures, we calculated Yue and Clayton dissimilarity matrices ($\theta$~YC~) in mothur [@Yue2005]. We rarefied samples to 2,107 sequences per sample to limit uneven sampling biases. We tested for differences in individual taxonomic groups that would explain the outcome differences with LEfSe [@Segata2011] in mothur. Remaining statistical analysis and data visualization was performed in R (v4.0.5) with the tidyverse package (v1.3.1). We tested for significant differences in $\beta$-diversity ($\theta$~YC~) using the Wilcoxon rank sum test. We used Spearman's correlation to identified which OTUs that had a correlation between their relative abundance and the histopathologic summary score. *P* values were then corrected for multiple comparisons with a Benjamini and Hochberg adjustment for a type I error rate of 0.05 [@Benjamini1995]. We built random forest models using the mikropml package [@Topcuoglu2021] with OTUs from day 0 samples using mtry values of 1 through 10, 15, 20, 25, 40, 50, 100. The split for training and testing varied by model to avoid overfitting the data. The toxin and moribundity models were trained with 60% of the data. The histopathologic score model was trained with 80% of the data. Lastly, we did not analyze murine communities to donor community or clinical data because germ-free mice colonized with human fecal communities become more like indigenous murine communities once stabilized. Furthermore, we are not trying to make any claims about human bacteria and their association with human CDI. 

**Code availability.** Scripts necessary to reproduce our analysis and this paper are available in an online repository (https://github.com/SchlossLab/Lesniak_Severity_XXXX_2022).

**Sequence data accession number.** All 16S rRNA gene sequence data and associated metadata are available through the Sequence Read Archive via accession PRJNA787941.

## Acknowledgements

Thank you to Sarah Lucas and Sarah Tomkovich for critical discussion in the development and execution of this project.
We also thank the University of Michigan Germ-free Mouse Core for assistance with our germfree mice, funded in part by U2CDK110768.
This work was supported by several grants from the National Institutes for Health R01GM099514, U19AI090871, U01AI12455, and P30DK034933. Additionally, NAL was supported by the Molecular Mechanisms of Microbial Pathogenesis training grant (NIH T32 AI007528).
The funding agencies had no role in study design, data collection and analysis, decision to publish, or preparation of the manuscript.

\newpage

## References

<div id="refs"></div>

\newpage

![](../results/figures/figure_1.jpg)

**Figure 1. Human fecal microbial communities established diverse gut bacterial communities in germ-free mice.** (A) Relative abundance of 10 most abundant bacterial classes of the gut bacterial communities on timepoint day 0, 14 days post-colonization of germ-free C57Bl/6 mice with human fecal samples. Each column of abundances represents an individual mouse. Each group of mice that received the same donor are grouped together and labeled with a letter (N for non-moribund mice and M for moribund mice) and number. (B) Median (points) and interquartile range (lines) of $\beta$-diversity ($\theta$~YC~) between an individual mouse and either all others which were inoculated with feces from the same donor or from a different donor. The $\beta$-diversity among the same donor comparison group was significantly less than the $\beta$-diversity of the different donor group (*P* < 0.05, calculated by Wilcoxon rank sum test).


\hfill\break

![](../results/figures/figure_2.jpg)

**Figure 2. All donor groups resulted in *C. difficile* infection but with different outcomes.** *C. difficile* CFU per gram of stool was measured the day after challenge with 10$^{3}$ *C. difficile* RT027 human isolate 431 spores and at the end of the experiment, 10 days post-challenge. Each point represents an individual mouse. Mice are grouped by donor and labeled by the donor letter (N for non-moribund mice and M for moribund mice) and number. Points are colored by donor group. Mice from donor groups N1 through N6 succumbed to the infection prior to day 10 and were not plated on day 10 post-challenge. LOD = Limit of detection.


\hfill\break

![](../results/figures/figure_3.jpg)

**Figure 3. Histopathologic score and toxin activity varied across donor groups.** (A) Fecal toxin activity was detected in some mice post *C. difficile* challenge in both moribund and non-moribund mice. (B) Cecum scored for histopathologic damage from mice at the end of the experiment. Samples were collected for histopathologic scoring on day 10 post-challenge for non-moribund mice or the day the mouse succumbed to the infection for the moribund group (day 2-3 post-challenge). Each point represents an individual mouse. Mice are grouped by donor and labeled by the donor letter (N for non-moribund mice and M for moribund mice) and number. Points are colored by donor group. Missing points are from mice that had insufficient sample collected for assaying fecal toxin.


\hfill\break

![](../results/figures/figure_4.jpg)

**Figure 4. Individual fecal bacterial community members of the murine gut associated with *C .difficile* infection outcomes.** (A and B) Relative abundance of genera at the time of *C. difficile* challenge (Day 0) that varied significantly by the moribundity and histopathologic summary score or toxin activity by LEfSe analysis. Median (points) and interquartile range (lines) are plotted. Genera are ordered alphabetically to ease comparisons across analyses. (A) Relative abundances were compared across infection outcome of moribund (colored black) or non-moribund with either a high histopathologic score (score greater than the median score of 5, colored green) or a low histopathologic summary score (score less than the median score of 5, colored light green). (B) Relative abundances were compared between mice which toxin activity was detected (Toxigenic, colored dark purple) and which no toxin activity was detected (Non-toxigenic, colored light purple). (C) Endpoint bacterial OTUs correlated with histopathologic summary score. Each individual mouse is plotted (transparent gray point).  Spearman's correlations were statistically significant after Benjamini-Hochberg correction for multiple comparisons. * next to a name indicates that the bacterial group was unclassified at lower classification ranks.


\hfill\break

![](../results/figures/figure_5.jpg)

**Figure 5. Fecal bacterial community members of the murine gut at the time of *C. difficile* (Day 0) predicted outcomes of the infection.** Day 0 bacterial community members grouped by different classification rank were modeled with random forest to predict the infection outcome. The models used the highest taxonomic classification rank that performed as well as the lower ranks. Median (solid points) and interquartile range (lines) of the group relative abundance are plotted. Bacterial groups are ordered by their importance to the model; taxonomic group at the top of the plot had the greatest decrease in performance when its relative abundances were permuted. * next to the name indicates that the bacterial group was unclassified at lower classification ranks. (A) Bacterial members grouped by phyla predicted which mice would have toxin activity detected at any point throughout the infection (Toxigenic, dark purple). (B) Bacterial members grouped by class predicted which mice would become moribund (dark blue). (C) Bacterial members grouped by genera predicted if the mice would have a high (score greater than the median score of 5, colored dark green) or low (score less than the median score of 5, colored light green) histopathologic summary score.


\hfill\break

![](../results/figures/figure_6.jpg)

**Figure 6. *C. difficile* infection outcome were similar within human fecal donor groups across different *C. difficile* isolates.** Mice colonized with human fecal donor communities N7, N6, or M4 were challenged with the isolate from the previous experiments in this study, *C. difficile* RT027 human isolate 431 (blue points), or a different *C. difficile* isolate (purple points). *C. difficile* CFUs were enumerated at the end of the experiment (day 10 post-challenge) and are plotted grouped by donor source (top plot). Histopathologic scores from the tissue collected at the endpoint (either day 10 post-challenge or day mice succumbed to infection) are plotted grouped by donor source (bottom plot). Each point represents an individual mouse. Mice that had received fecal communities from human donor M4 succumbed to the infection prior to day 10 post-challenge so were not plated for CFU on day 10 post-challenge (DECD = deceased). RT = ribotype, Isolate = *C. difficile* human isolate id, LOD = limit of detection.


\hfill\break

![](../results/figures/figure_S1.jpg)

**Figure S1. Histopathologic score of tissue damage at the endpoint of the infection.** Tissue collected at the endpoint, either day 10 post-challenge (Non-moribund) or day mice succumbed to infection (Moribund), were scored from histopathologic damage. Each point represents an individual mouse. Mice (points) are grouped and colored by their human fecal community donor. Missing points are from mice that had insufficient sample for histopathologic scoring.

\hfill\break

![](../results/figures/figure_S2.jpg)

**Figure S2. Random forest models predicted outcomes of the *C. difficile* challenge.** (A-C) Taxonomic classification rank model performance. Relative abundance at the time of *C. difficile* challenge (Day 0) of the bacterial community members grouped by different classification rank were modeled with random forest to predict the infection outcome. The models used the highest taxonomic classification rank that performed as well as the lower ranks. Black rectangle highlights classification rank used to model each outcome. (D-F) Model feature importance. Bacterial groups are ordered by their decrease in area under receiver-operator curve (AUC) when its relative abundances was permuted. Individual relative abundances were added to F since differences in AUC were outside the interquartile range. * indicates bacterial group was unclassified at lower classification ranks. For all plots, median (solid points) and interquartile range (lines) are plotted. (A) Toxin production modeled which mice would have toxin detected during the experiment. (B) Moribundity modeled which mice would succumb to the infection prior to day 10 post-challenge. (C) Histopathologic score modeled which mice would have a high (score greater than the median score of 5) or low (score less than the median score of 5) histopathologic summary score. (D) Bacterial phyla which affected the performance of predicting toxin activity when permuted. (E) Bacterial classes which affected the performance of predicting moribundity when permuted. (D) Bacterial genera which affected the performance of predicting histopathologic score when permuted.

\hfill\break

![](../results/figures/figure_S3.jpg)

**Figure S3. Subset of donor communities were selected to test for conserved *C. difficile* infection outcome with a different *C. difficile* isolate.** (A) Relative abundance of genera in mice from previous experiments which received human fecal donor communities N7, N6, or M4. Each column represents an individual mouse. (B) Features from the random forest model that predicted moribundity using relative abundance of bacteria were grouped by their genus classification rank and are ordered by their decrease in AUC when each feature was permuted. Median (solid points) and interquartile range (lines) change in AUC are plotted. (C) Relative abundance of the most important genera used by the model to predict moribundity. Median (solid points) and interquartile range (lines) of the group relative abundances are plotted. * next to a name indicates that the bacterial group was unclassified at lower classification ranks.

