---
csl: mbio.csl
fontsize: 12pt
output:
  pdf_document:
    includes:
      in_header: header.tex
    keep_tex: yes
  word_document: default
bibliography: references.bib
---


```{r knitr_settings, eval=TRUE, echo=FALSE, cache=FALSE, warning=FALSE, message=FALSE}
library(knitr)
library(readr)

opts_chunk$set("tidy" = TRUE)
opts_chunk$set("echo" = FALSE)
opts_chunk$set("eval" = TRUE)
opts_chunk$set("warning" = FALSE)
opts_chunk$set("cache" = FALSE)
opts_chunk$set("message" = FALSE)

```

\pagenumbering{arabic}
\linenumbers
\doublespacing

# The gut bacterial community potentiates *Clostridiodes difficile* infection severity.
  


\vspace{30mm}


**Running title:** Microbiota potentiates *Clostridioides difficile* infection severity

\vspace{20mm}

Nicholas A. Lesniak$^1$, Alyxandria M. Schubert$^1$, Kaitlyn Flynn$^1$, Jhansi L Leslie$^1$, Hamide Sinani$^1$, Ingrid L. Bergin$^3$, Vincent B Young$^2$, Patrick D. Schloss$^{1,\dagger}$

\vspace{30mm}

$\dagger$ To whom correspondence should be addressed: pschloss@umich.edu  
1\. Department of Microbiology and Immunology, University of Michigan, Ann Arbor, MI  
2\. Division of Infectious Diseases, Department of Internal Medicine, University of Michigan Medical School, Ann Arbor, MI
3\. Unit for Laboratory Animal Medicine, University of Michigan, Ann Arbor, MI




\newpage 


## Abstract

*Clostridioides difficile* infection (CDI) has become more common and severe in the last few decades. Patient age, white blood cell count, creatinine levels as well as *C. difficile* ribotype and the presence of toxin genes have been associated with disease severity. However, current models of CDI disease severity that incorporate these factors are not robust enough to be broadly applied. These models are missing input from the gut environment. CDI is dependent on perturbations of the gut bacterial community in order to colonize the gut. The gut microbiota also impairs *C. difficile* colonization through bile acid metabolism, nutrient consumption and bacteriocin production. However, it is unclear if the gut bacteria affect CDI disease severity. Here we demonstrate that gut bacterial communities contribute to disease severity variation. We derived diverse gut communities by colonizing germ-free mice with human fecal communities. The mice were then infected with a single *C. difficile* RT027 isolate which resulted in mortality and histopathologic differences. The severity grouped by the human fecal community they received. Generally, facultative anaerobes with pathogenic potential, such as *Escherichia*, *Helicobacter*,  and *Klebsiella*, were associated with more severe disease. Bacterial groups associated with dietary fiber degradation, such as *Coprobacillus*, were associated with reduced disease severity. Lastly, we showed communities that resulted in either low or high histopathologic scores or severe disease had the same outcome when infected with a different *C. difficile* isolate.

## Importance

*Clostridioides difficile* infection (CDI) can be asymptomatic, or range in severity from mild diarrhea to severe outcomes of surgical interventions and even death. Models that predict severity and guide treatment decisions are based on clinical factors and *C. difficile* characteristics. The gut microbiome plays a role in potecting against CDI, though has not been investigated for its effect on CDI disease severity. We demonstrate that variation in the microbiome of mice colonized with human feces was sufficient to result in a range of disease severities. The results reveal groups of bacteria associated with both severe and more mild *C. difficile* infections. Gut bacterial community data from patients with CDI could improve our ability to identify patients at risk for severe disease and improve interventions which treat the gut bacteria to reduce host damage.



\newpage


## Introduction

*Clostridioides difficile* infections (CDI) have been increasing in incidence and severity since *C. difficile* was first identified as the cause of antibiotic-associated psuedomembranous colitis [@Kelly2008]. Disease severity can range from asymptomatic to mild diarrhea to toxic megacolon and death. Treatment for CDI is driven by the disease severity. The Infectious Diseases Society of America (IDSA) and Society for Healthcare Epidemiology of America (SHEA) guidelines categorize severe disease based on white blood cell count >15,000 cells/mm^3 and/or a serum creatinine >1.5 mg/dL, and complicated disease based on systolic blood pressure <90, ileus or toxic megacolon [@McDonald2018]. Since these measures are only present once the disease has advanced, the IDSA classification does not allow for identification of patients at risk of severe CDI. Thus, we have limited ability to prevent patients from developing severe CDI. Ideally, we could identify patients that are high-risk of severe disease use that to adjust their treatments to halt the disease before it worsens.

Prediction models have been developed to score a patients risk for severe CDI outcomes but have not been robust for broad application [@Dieterle2020; @Butt2013; @Perry2021; @vanBeurden2017]. Firstly, there is no widely accepted definition of severe disease [@Ressler2021]. While the IDSA guideline is frequently used, many studies develop their own severity definitions. Aside from the features utilized by the IDSA guidelines, other severity measures use patient demographics, vital information, laboratory test results, disease complications, or infection outcomes such as ICU admission, surgery and/or death [@Dieterle2020; @Zar2007; @Gujja2009; @Jardin2013]. However, a recent multi-center external validation of these scoring systems using a consistent severity definition - transfer or admission to the intensive care unit, colectomy, and/or death within 30 days of diagnosis - demonstrated a decline in performance and high false positivity rate [@Perry2021]. Thus, it may be necessary to incorporate information in addition to patient and clinical factors to accurately idenitfy patients at risk of severe CDI.  

CDI disease severity is the result of damage to the host. Severity has been associated with characteristics of *C. difficile* ribotypes, such as 027, and host factors [@Bauer2011]. Ribotypes with genes for toxin TcdA/TcdB have the potential to disrupt the tight junctions between the intestinal epithelial cells [@Abt2016]. The presence of binary toxin CDT or mutations in *tcdC*, the negative regulator of toxin transcription, have been associated with increased virulence of TcdA/TcdB [@Abt2016]. In addition to the pathogenic potential of *C. difficile*, the response by the host also affects the disease severity. IgG antibody activity towards the TcdA/TcdB are decreased in both patients with symptomatic and recurrent infections [@Kelly2011]. In addition to controlling the infection, the immune response can also cause intestinal injury. Increased neutrophil inflitration into the mucosa after the cellular tight junctions have been compromised can exacerbate the inflammatory response [@Farooq2015]. While we have have elucidated these major drivers of host damage in CDI, these do not explain the observed variation in disease severity.

Missing from our CDI disease model are the effects of gut bacteria. *C. difficile* interacts with the intestinal community in many ways. The indigenous bacteria of a healthy intestinal community provide a protective barrier preventing *C. difficile* from infecting the gut. A range of mechanisms can disrupt this barrier, including antibiotics, medications, or dietary changes, and lead to increased susceptiblity to CDI [@Britton2014; @Hryckowian2018; @VichVila2020]. Once in the gut, the indigenous bacteria can either promote or inhibit *C. difficile* through producing molecules or modifying the environment [@Abbas2020]. Bile acids metabolized by the gut bacteria can inhibit *C. difficile* growth and affect toxin production [@Sorg2008; @Thanissery2017]. Bacteria in the gut also can compete more directly with *C. difficile* through antibiotic production or nutrient consumption [@Aguirre2021; @Kang2019]. While the interaction between the gut bacteria and *C. difficile* is established, the effect the gut bacteria on CDI disease severity is unclear. Recent studies that have begun to uncover this relationship but have either used clinical data which does not permit control of the host or *C. difficile* variation [@Seekatz2016] or murine experiments which lack variation in the bacterial communities [@Dieterle2020]. Recently, Tomkovich et al. showed the variation in the bacterial communities between mice from different mouse colonies and vendors resulted in different clearance rates of *C. difficile* [@Tomkovich2020]. Therefore it is clear the gut bacteria modulate the activity and infection dynmaics of *C. difficile*, we hypothesized that gut bacteria also contribute to the variation in CDI disease severity. Here, we colonized germ-free C57BL/6 mice with human fecal communities to test if the variablity in the gut bacteria can produce and explain variation in the CDI disease severity.

## Results

***C. difficile* is able to infect germ-free mice colonized with human fecal microbial communities without perturbation.**  Based on our previous observation that variation in the murine microbiota associated with different *C. difficile* colonization dynamics [@Tomkovich2020], we sought to explore how variation in murine microbial communities affects the outcome of *C. difficile* infection. To produce communities with greater variation than mouse colonies with their indigenous communties, we used human fecal communities to establish a microbial community in the intestine of germ-free mice [@Nagpal2018]. We inoculated germ-free C57BL/6 mice with homogenized feces from one of 15 human fecal samples via oral gavage. The human fecal samples selected were chosen to provide diverse communtiy structures and based on community clusters [@Schubert2014]. The gut community was allowed to establish itself for two weeks post inoculation [@Gillilland2012]. We then surveyed the bacterial members of the gut communities by 16S rRNA gene sequencing of fecal pellets (Figure 1A). The bacterial communities from each mouse grouped more closely to those communities from mice that received the same human fecal donor community than to either their respective donor community or any mice who received a fecal homogenate from a different donor (Figure 1B). Most of the communities were primairily composed of populations of *Clostridia*, *Bacteroidia*, *Erysipelotrichia*, *Bacilli*, and *Gammaproteobacteria*. However, each group of mice harbored unique combination of class abundances in their gut bacterial communities.

Next, we tested this set of mice with their human-derived gut microbial communities for susceptibility to *C. difficile* infection. A typical mouse model of CDI requires pretreatment with antibiotics such as clindamycin to become susceptible to *C. difficile* colonization [@Chen2008; @Schubert2015]. However, we wanted to avoid modifying the communties with an antibiotic to maintain their unique microbial compositions and ecological relationships. Since these communties came from people who have increased risk of CDI, such as hospitalization or recent antibiotic use [@Schubert2014], or the communities became susceptible through the murine gut colonization process, we decided to test if *C. difficile* was able to infect these mice without any antibiotic perturbation. We hypothesized that *C. difficile* would be able to colonize a subset of mice whose communities were susceptible from perturbations either the donor experienced or from the ecological selection colonizing the murine gut. After two weeks post incoluation with human fecal communities, the mice were then challenged with $10^{3}$ *C. difficile* ribotype 027 human isolate 431 spores. The mice were followed for 10 days post-challenge and their stool was collected and plated for *C. difficile* colony forming units (CFU) to detect infection level. Suprisingly, communities from all donors were colonized (Figure 2). Only two mice were able to resist *C. difficile* infection and both received their community from the same donor, Donor C. The remaining mice demonstrated susceptiblity to *C. difficile* infection without the need for any perturbation. Thus, *C. difficile* infected the mice without needing to perturb the established human-derived gut microbial communities, allowing them to maintain their unique community structures. 

**Infection severity varies by initial community.** After challenging the mice with *C. difficile*, we investigated the outcome from the *C. difficile* infection and its relationship to the initial community. We followed the mice for 10 days post-challenge for *C. difficile* CFU, toxin production, and mortality. Seven mice, from Donors A, B and E, were not initally colonized on the day after *C. difficile* challenge but were infected ($>10^{6}$) by the end of the experiment. All mice that received their community from Donor J through O suffered from severe disease, which became highly colonized one day post-challenge and moribund within 3 days post-challenge. The remaining mice, except the Donor C mice which were not infected, maintained *C. difficile* infection through the end of the experiment (Figure 2). At 10 days post-challenge, or earlier for the moribund mice, mice were euthanised and cecal tissue was collected and scored for histopathologic signs of disease and fecal pellets were assayed for toxin activity (Figure 3). Overall, there was greater toxin activity detected in the stool of the moribund mice (*P* = 0.003). However, when looking at each group of mice, we saw there was a range in toxin activity for both the moribund and non-moribund mice (Figure 3A). Non-moribund mice from Donors D through I had comparable toxin activity as the moribund mice. Additionally, not all moribund mice had toxin activity detected in their stool. This discordance between toxin activity and disease severity has been previously described [@Akerlund2006; @Vitucci2020]. Next we examined the cecal tissue for histopathologic damage. For moribund mice, we observed high levels of epithelial damage, tissue edema and inflammation (Figure S1), similar to previously reported histopathologic findings for *C. difficile* ribotype 027 [@Cowardin2016]. As observed with toxin activity, the moribund mice had higher histopathologic scores than the mice with non-severe disease (*P* = 3.0e-9). However, unlike the toxin activity, all moribund mice had consistently high histopathologic summary scores (Figure 3B). For Donor groups A through I, where all mice were non-moribund, we saw a range in tissue damage from no signs of disease to nearly the same level as the mice with severe disease, which grouped by community donor. Together, the toxin activity, histopathologic score, and moribundity showed variation across the donors but were largely consistent within each group of mice recieving the same human donor fecal community.

**Microbial community members explain variation in CDI severity.** Since the disease charateristics grouped by human donor fecal community, we next investigated specific bacterial populations that could explain the variation in disease severity. To this aim, we interogated the day 0 bacterial community for their relationship to our CDI severity measurements using linear discriminant analysis (LDA) effect size (LEfSe) analysis. We split the mice into groups by severity level based on their moribundity and histopathologic score. We dichotomized the histopathologic scores into high and low bins by splitting on the median score. This analysis revealed 20 genera that were significantly different by the disease severity (Figure 4A). Bacterial genera *Turicibacter*, *Streptococcus*, *Staphylococcus*, *Pseudomonas*, *Phocaeicola*, *Parabacteroides*, *Bacteroides*, and *Escherichia/Shigella* were detected at higher relative abundances in the mice that became moribund. Populations of *Anaerotignum*, *Coprobacillus*, *Enterocloster*, and *Murimonas* were more abundant in the mice that would develop only low intestinal injury. To understand the role of toxin activity in disease severity, we applied LEfSe to identify the genera most likely to explain the observed differences in toxin activity (Figure 4B). Many genera the were associated with toxin production as we would expect from their association with moribundity, such as populations of *Escherichia/Shigella* and *Bacteroides*. Likewise, there were genera such as *Anaerotignum*, *Enterocloster*, and *Murimonas* that exhibited greater relative abundance in communities from non-moribund mice with a low histopathologic score and no detected toxin. Overall, communities without genera that demonstrate a linear trend between relative abundance and disease severity can be better understood by their association with toxin activity. Lastly, we tested for correlations between the histopathologic score and the endpoint relative abundances of bacterial operational taxonomic units (OTUs) (Figure 4C). The endpoint relative abundance of *Bacteroides* was positively correlated with histopathologic score, as its day 0 relative abundance did with disease severity (Figure 4A). Populations of *Klebsiella* and *Prevotellaceae* were positively correlated with the histopathologic score and were increased in the group of mice with detectable toxin. Our analysis have revealed bacterial taxa that are associated with three different measures of CDI severity in our model: moribundity, histopathologic score, and toxin activity.

We next wanted to determine whether collectively, bacterial community membership and relative abundance could be predictive of CDI severity. We trained random forest models with the day 0 bacterial community relative abundance data at each taxonomic rank to predict toxin production, severe disease or final histopathologic score. Overall for predicting toxin production, microbial populations aggregated by their phylum level classificiation performed as well as models using lower taxonomic ranks (AUROC = 0.83, Figure S2). Collectively, members of *Verrucomicrobia*, *Campilobacterota* and *Proteobacteria* contributed the most to the correct prediction of toxin production (Figure 5A). *C. difficile* was more likely to produce toxin when the community it infected had less abundant populations of *Verrucomicrobia* and *Campilobacterota* and had more abundant populations of *Proteobacteria*. Next, we assessed the ability of the community to predict moribundity. Class rank classification was sufficient to predict if the mouse would succumb to the infection before the end of the experiment (AUROC = 0.91, Figure S2). Bacterial taxa belonging to *Bacilli*, *Firmicutes*, and *Erysipelotrichia* hasd the greatest effect on model accuracy. Larger populations of *Bacilli* and *Firmicutes* and reduced populations of *Erysipelotrichia* were more likely to result in moribundity (Figure 5B). The only other class that was decreased in moribund mice was *Clostridia*. Lastly, the relative abundances of genera were able to predict a high or low histopathologic score (histopathologic score dichotomized as in previous analysis, AUROC = 0.99, Figure S2). No genera had a significantly greater effect on the model performance than any others, indicating the model is reliant on many genera for the correct prediction. The model used some of the genera identified in the LEfSe analysis, such as *Coprobacillus*, *Anaerostipes*, and *Hungatella*. Communities with greater abundances of *Hungatella*, *Eggerthella*, *Bifidobacterium*, *Duncaniella* and *Neisseria* were more likely to have high histopathologic scores. 

**Disease severity consistent by donor community across isolates.** Since Carlson et al. showed clinical disease severity varied by *C. difficile* isolate characteristics [@Carlson2013; @Carlson2015], we next wanted to test if the effect of the community on CDI severity is conserved across *C. difficile* isolates. To facilitate this analysis, we selected a subset of donor fecal samples based on their associated mouse transplant communities that fit the following criteria: one community we would expect to result in a low histopathologic score (< 5), a high histopathologic score (> 5), and a high histopathologic score which becomes moribund. Additionally, we selected the donors based their relative abundances of genera identified through the our analysis (Figure S3). We inoculated mice via oral gavage of the human fecal slurry of either Donor F, G, or M and allowed two weeks for the a community to establish. We then challenged the mice with a different *C. difficile* isolate and all mice were infected. The moribundity and histopathologic score outcomes from challenge with the second *C. difficile* human isolate were comparable to the outcomes of challenges with *C. difficile* ribotype 027 human isolate 431 by their donor group (Figure 6).


## Discussion

Challenging mice colonized with different human fecal communities with a single *C. difficile* isolate allowed us to identify the effect of microbiome variation on *C. difficile* infection disease severity. Our LEfSe analyses and random forest models revealed the relationships between the bacterial community members and the degree of disease severity. We used the relative abundance patterns of community members with strong associations with disease severity to select communities to test if the observed disease severity is conserved when a second *C. difficile* isolate is used. The disease outcome from the *C. difficile* chalenge with the new isolate matched the expected disease severities. Therefore, the effect of the community variation on disease severity *in vivo* is greater than the effect of *C. difficile* isolate phenotype variation [@Carlson2013; @Carlson2015]. Overall, these results have demonstrated that the bacterial community contributes to the disease severity of *C. difficile* infection.

CDI severity can range from no symptoms, mild symptoms such as diarrhea, or severe symptoms such as toxic megacolon or death. Physicians use prediction tools to identify patients most at risk of developing a severe infection using white blood cell counts, serum albumin level, or serum creatine level [@Lungulescu2011; @McDonald2018; @Zar2007]. Those levels are being driven by the activities in the intestine [@diMasi2018]. Research into the drivers of this variation have revealed factors that make *C. difficile* more virulent. Strains are categorized for their virulence by the presence and production toxin A, toxin B, and binary toxin and the prevelance in outbreaks, such as ribotype 027 and 078 [@AbernathyClose2020; @Theriot2011; @Goorhuis2008; @OConnor2009; @Chen2008]. However, other studies have shown that virulence is not necessarily linked with toxin production [@Rao2015] or the strain [@Walk2012]. Furthermore, there is variation in the genome, growth rate, sporulation, germination, and toxin production in different isolates of a strain [@Carlson2013; @He2010]. This variation may help explain why severe CDI prediction tools often miss identifying many patients with CDI that will develop severe disease [@Dieterle2020; @Butt2013; @Perry2021; @vanBeurden2017]. Therefore, it is necessary to gain a full understanding of all factors contributing to disease variation to improve our ability to predict severity.

The state of the intestinal bacterial community determines the ability of *C. difficile* to colonize, persist and even cause recurrent infections. *C. difficile* is unable to colonize an unperturbed healthy gut community and is only able to become established after a perturbation [@Schubert2015]. Once colonized, the different communities lead to different metabolic responses and dynamics of the *C. difficile* population [@Jenior2017, Jenior2018; @Lesniak2021]. The gut bacteria metabolize primary bile acids into secondary bile acids [@Staley2016; @Long2017]. The concentration of these bile acids affect the germination, growth, toxin production and biofilm formation [@Sorg2008; @Sorg2010; @Thanissery2017; @Dubois2019]. Members of the bacterial community also affect other metabolites *C. difficile* utilizes. *Bacteroides thetaiotaomicron* produce sialidases which release sialic acid from the mucosa for *C. difficile* to utilize [@Ng2013; @Ferreyra2014]. The nutrient environemt affects toxin production [@MartinVerstraete2016]. Thus, many of the actions of the gut bacteria modulate *C. difficile* in ways that would affect the disease severity driven by CDI.

Myriad studies have explored the relationship between the microbiome and CDI but none have established a relationship between the variation of disease severity and the microbiome. Most CDI studies employ either an *in vitro* or *in vivo* model using a homogenous bacterial community. Collins et al. used multiple human communities to colonize mice, however the communities were pooled prior to gavaging into germ-free mice [@Collins2015], resulting in a single community. Studies examining difference in disease often use different *C. difficile* strains or ribotypes in mice with similar microbiota as a proxy for variation in disease, such as strain 630 for non-severe and ribotype 027 for severe [@AbernathyClose2020; @Theriot2011; @Lawley2012; @Chen2008]. There have also been studies demonstrating variation in severity through tapering antibiotic dosage [@Lesniak2021; @Schubert2015; @Reeves2011] or by reducing the amount of cells or spores used for the challenge [@Chen2008; @Reeves2011]. Our group has recently been uncovering how variation in the microbiome affects CDI but have been limited to *C. difficile* colonization [@Lesniak2021; @Tomkovich2020]. 

With our recent observations that the initial community affects the ability of *C. difficile* to persist in an environment and the existing research describing the myriad ways the microbiome interacts with *C. difficile*, we hypothesized that the microbiome directly and indirectly modulates CDI disease severity. Our data demostrate gut bacterial relative abundances associate with variation in toxin production, histopathologic scoring of the cecum and mortality. This anaylsis revealed populations of *Akkermansia*, *Anaerostipes*, *Coprobacillus*, *Enterocloster*, *Lactonifactor*, and *Monoglobus* were more abundant in the microbiome of mice which had low histopathologic scores and no mortality or detected toxin. The protective role of these genera are supported by previous studies. *Coprobacillus*, *Lactonifactor*, and *Monoglobus* have been shown to be involved in dietary fiber fermentation and associated with healthy communities [@Mabrok2011; @Kim2019; @doPrado2021; @Muthuramalingam2019]. *Anaerostipes* and *Coprobacillus* produce short chain fatty acids which also have been associated with healthy communities [@Han2019; @Duncan2004; @Ye2018]. Furthermore, *Coprobacillus*, which is abundant in mice with low histopathologic scores but rare in all other mice, has been shown to contain putative type I lantibiotic gene cluster and inhibit *C. difficile* colonization [@Walsh2017; @Sandiford2018; @Stein2013]. *Akkermansia* and *Enterocloster* were also idenitified as more abundant in mice which had a low histopathologic scores but have contradictory supporting evidence in the current literature. In our data, *Akkermansia* is most abundant in the non-moribund mice with low histopathologic scores but there were some moribund mice which had increased populations of *Akkermansia*. This could be attributed to either a more protective mucus layer was present inhibiting colonization [@Stein2013; @Nakashima2021] or mucus consumption by *Akkermansia* could have been crossfeeding *C. difficile* or exposing a niche for *C. difficile* [@Geerlings2018; @Deng2018; @Engevik2020]. Similarly with *Enterocloster*, in our data this genus was more abundant and associated with low histopathologic scores. It has been associated with healthy populations and has been used to mono-colonize germ-free mice to reduce the ability of *C. difficile* to colonize [@Reeves2012; @Ma2021]. However, *Enterocloster*, a member of indigenous gut communities, has been involved in infections, such as bacteremia [@Haas2020; @Finegold2005]. These data demonstrate a population of bacteria has the potential to be either protective or harmful. The disease outcome is not likely based on the abundance of a single population of bacteria rather the result of the interactions of many populations.

The groups of bacteria that were associated with either a higher histopathologic score or moribundity are members of the indigenous gut community that also have been associated with disease, often referred to as opportunitic pathogens. Many of the populations with pathogenic potential that associated with worse outcomes are also facultative anaerobes. *Enterococcus*, *Klebsiella*, *Shigella/Escherichia*, *Staphylococcus*, and *Streptococcus* have been shown to expand after antibiotic use [@VanInsberghe2020; @GarzaGonzalez2019; @Schubert2014] and are commonly detected in CDI cases [@Shafiq2020; @Keith2020; @Zackular2016; @Berkell2021]. In addition to these populations, *Eggerthella*, *Prevotellaceae* and *Helicobacter*, associated with worse outcomes have also been associated with intestinal inflammation [@Gardiner2014; @Iljazovic2020; @Nagalingam2013]. Recently, *Helicobacter hepaticus* has been demostrated to be sufficient to cause susceptibility to CDI in IL-$10^{-/-}$ C57BL/6 mice [@AbernathyClose2021]. In our experiments, when *Helicobacter* was present, the mouse resulted in a high histopathologic score. While we did not use IL-$10^{-/-}$ mice, it is possible the bacterial communtity or host response are similarly modified by *Helicobacter* allowing *C. difficile* infection and resulting in host damage. Aside from *Helicobacter*, these groups of bacteria associated with more severe outcomes did not have a conserved association between their relative abundance and the disease severity across all mice.

The relationship between a group of bacteria and severity may seem to vary, until we evaluate in the context of the environment inwhich there interact. Based on our results, it is more appropriate to apply the damage-response framework for microbial pathogenesis to CDI [@Pirofski2008; @Casadevall2014]. Disease is not driven by a single entity, rather it is an emergent property of the responses of the host immune system, infecting microbe, *C. difficile*, and the microbes of the indigenous community. In the first set of experiments, we use the same host, C57BL/6 mice, the same infecting microbe, *C. difficile* ribotype 027 human isolate 431, with different gut bacterial communities. The bacterial groups in those communities were often present in both moribund and non-moribund and across the range of histopathologic scores. Thus, it is not merely, the presence of the bacteria but their activity in response to the other microbes and host which affect the extent of the host damage. Additionally, while each mouse and *C. difficile* has the same genetic background, they too are reacting to the specific microbial community. Disease severity is driven by the cummulative effect of the host immune response and the activity of the gut bacteria and *C. difficile*. *C. difficile* can be driving host damage through the production of toxin. The gut microbiota could be contributing to the host damage through the balance of metabolic and competitive activities, such as bacteriocin production or mucin degradation. Low levels of mucin degradation can provide nutrients to other community members producing a diverse non-damaging community []. However, if mucin degradation becomes too great it reduces the protective function of the mucin layer and exposes the epithelial cells []. This over-harvesting contributes to the host damage due to other members producing toxin []. Thus, the resultant intestinal damage is the balance of all activities in the gut environment. Host damage is the emergent property of numerous damage-response curves, one for host immune response, one for *C. difficile* activity and another for microbiome community activity, each of which are a composite curve of the individual activities from each group, such as antibody production, neutrophil infiltration, toxin production, sporulation, fiber and mucin degradation. Therefore, while we have identified populations of interest, it is likely necessary to target multiple to reduce those with community interactions contributing to host damage.

Many studies have investigated individual activities contributing to CDI. Here we have shown myraid bacterial groups and their relative abundances associate with variation in CDI disease severity. We must continue to explore how the host, microbiota, and *C. difficile* interactions result in severe disease. We can reduce the risk of severity with a better understanding of which interactions, whether driven by specific community functions or groups of bacteria, reduce host damage. Approaching this problem from a damage-response framework allows us to experiment with modifying subsets of activities to reduce overall host damage. Our current clinical treatment for primary CDI targets an individual group of this tripartite system. Most commonly, CDI is treated with an antibiotic to eliminate *C. difficile*. Alternatively, another treatment uses antibodies to neutralize *C. difficile* toxins. These treatments are only addressing a single entity of the system, leaving the potential for the others to maintain responses that maintain host damage. However, we can continue this research to elucidate treatments that drive all three entities towards reduced host damage. When a patient is diagnosed with CDI, the gut community composition, in addition to the traditionally obtained information, may improve our severity prediction and appropriate treatment. Treating the microbiome at the time of diagnosis may prevent the infection from becoming more severity. Based on the general trends of our results, promoting fiber metabolizing bacteria and reducing facultative anerobes could bolster the efficacy of current CDI treatments, reducing disease severities.


## Materials and Methods

**Animal care.** 6- to 13-week old male and female germ-free C57BL/6 were obtained from a single breeding colony in the University of Michigan Germ-free Mouse Core. Mice were housed in cages of 2-4 mice per cage and maintained in germ-free isolators at the University of Michigan  germ-free facility. All mouse experiments were approved by the University Committee on Use and Care of Animals at the University of Michigan. **Were the mice maintained in gnotobiotic isolators throughout the experiment, one per donor? Or were they transferred to BSL2 conditions prior to administering the human FMTs?**

***C. difficile* experiments.** Human fecal samples were obtained from Schubert et al. and selected based on community clusters [@Schubert2014] to result in diverse community structures. Feces were homogenized by mixing 200 mg of sample with 5 ml of PBS. Mice were inoculated with 100 $\mu$l of the fecal homogenate via oral gavage. Two weeks after the fecal community inoculation, mice were challenged with *C. difficile*. *C. difficile* isolates came from Carlson et al. which had previously been isolated and characterized [@Carlson2013; @Carlson2015]. Spores concentration were determined both before and after challenge [@Sorg2009]. $10^{3}$ *C. difficile* spores were given to each mouse via oral gavage. **How were human FMTs prepared (anaerobically vs aerobically, etc)?**

**Sample collection.** Fecal samples were collected on the day of *C. difficile* challenge and the following 10 days. Each day, a fecal sample was also collected and weighed and the remaining sample was frozen at -20$^\circ$C. Anaerobically, the weighed fecal samples were serially diluted in PBS, plated on TCCFA plates, and incubated at 37$^\circ$C for 24 hours. The plates were then counted for the number of colony forming units (CFU) [@Winston2016].

**DNA sequencing.** From the frozen fecal samples, total bacterial DNA was extracted using MOBIO PowerSoil-htp 96-well soil DNA isolation kit. We amplified the 16S rRNA gene V4 region and sequenced the resulting amplicons using an Illumina MiSeq as described previously [@Kozich2013].

**Sequence curation.** Sequences were processed with mothur(v.1.44.3) as previously described [@Kozich2013; @Schloss2009]. In short, we used a 3% dissimilarity cutoff to group sequences into operational taxonomic units (OTUs). We used a naive Bayesian classifier with the Ribosomal Database Project training set (version 18) to assign taxonomic classifications to each OTU [@Wang2007]. We sequenced a mock community of a known community composition and 16s rRNA gene sequences. We processed this mock community with our samples to calculate the error rate for our sequence curation, which was an error rate of 0.19%.

**Toxin cytotoxicity assay.** Fecal sample remaining after plating for CFU and DNA extraction were collected in a preweighted sterile tube and stored at −80°C. To prepare the sample for the activity assay, samples were diluted 1:10 weight per volume using sterile PBS and then filter sterilized through a 0.22-$\mu$m filter. Toxin activity was assessed using a Vero cell rounding-based cytotoxicity assay as described previously [@Theriot2011]. The cytotoxicity titer was determined for each sample as the last dilution, which resulted in at least 80% cell rounding. Toxin titers are reported as the log10 of the reciprocal of the cytotoxicity titer.

**Histopathology evaluation.** Mouse ceca tissue was placed in histopathology cassettes and fixed in 10% formalin, then stored in 70% ethanol. McClinchey Histology Labs, Inc. (Stockbridge, MI) embedded the samples in paraffin, sectioned, and created the hematoxylin and eosin-stained slides. The slides were scored using previously described criteria by a board-certified veterinary pathologist scored who was blinded to the experimental groups [@Theriot2011].

**Statistical analysis and modeling.** To compare community structures, we calculated Yue and Clayton dissimilarity matrices in mothur [@Yue2005]. We rarified samples to 2,107 sequences per sample to limit uneven sampling biases. We tested for differences in individual taxonomic groups that would explain the outcome differences with LEfSe [@Segata2011] in mothur. Remaining statistical analysis and data visualization was performed in R (v4.0.5) with the tidyverse package (v1.3.1). We tested for significant differences in $\beta$-diversity ($\theta$~YC~)using the Wilcoxon rank sum test. We used Spearman's corrlation to identified the OTU which had a correlation between their relative abundance and the histopathologic summary score. *P* values were then corrected for multiple comparisons with a Benjamini and Hochberg adjustment for a type I error rate of 0.05 [@Benjamini1995]. We built random forest models using the mikropml package [@Topcuoglu2021] with OTUs from day 0 samples using mtry values of 1 through 10, 15, 20, 25, 40, 50, 100. The split for training and testing varied by model to avoid overfitting the data. The toxin and moribundity models were trained with 60% of the data. The histopathologic score model was trained with 80% of the data. Lastly, we did not analyze murine communities to donor community or clinical data because germ-free mice colonized with human fecal communities become more like indigenous murine communities once stabilized. Futhermore, we are not trying to make any claims about human bacteria and their association with human CDI. 

**Code availability.** Scripts necessary to reproduce our analysis and this paper are available in an online repository (https://github.com/SchlossLab/Lesniak_Severity_XXXX_2021).

**Sequence data accession number.** All 16S rRNA gene sequence data and associated metadata are available through the Sequence Read Archive via accession XXXX.

## Acknowledgements

Thank you to Sarah Lucas and Sarah Tomkovich for critical discussion in the development and execution of this project.
We also thank the University of Michigan Germ-free Mouse Core for assitance with our germfree mice, funded in part by U2CDK110768.
This work was supported by several grants from the National Institutes for Health R01GM099514, U19AI090871, U01AI12455, and P30DK034933. Additionally, NAL was supported by the Molecular Mechanisms of Microbial Pathogenesis training grant (NIH T32 AI007528).
The funding agencies had no role in study design, data collection and analysis, decision to publish, or preparation of the manuscript.

\newpage

## References

<div id="refs"></div>

\newpage

![](../results/figures/figure_1.jpg)

**Figure 1. Human fecal microbial communtities establish diverse gut bacterial communties in germ-free mice.** (A) Relative abundace of 10 most abundant bacterial classes of the gut bacterial communities on timepoint day 0, 14 days post-inoculation of germ-free C57Bl/6 mice with human fecal samples. Each column of abundances represent an individual mouse. Each group of mice that recieved the same donor are group together and labeled with a letter (A through O). (B) Median (points) and interquartile range (lines) of $\beta$-diversity ($\theta$~YC~) between an individual mouse and either all others which were inoculated with feces from the same donor or from a different donor. The $\beta$-diversity amongst in the same donor comparison group was significantly less than the $\beta$-diversity of the different donor group (*P* < 0.05, calculated by Wilcoxon rank sum test).


\hfill\break

![](../results/figures/figure_2.jpg)

**Figure 2. All donor groups resulted in *C. difficile* infection but with different outcomes.** *C. difficile* CFU per gram of stool measured the day after challenge with $10^{3}$ *C. difficile* ribotype 027 human isolate 431 spores and at the end of the experiment, 10 days post-challenge. Each point is an individual mouse. Mice are grouped by donor and labeled by the donor letter. Points are colored by donor group. Mice from donor groups J through O succumbed to the infection prior to day 10 and were not plated. LOD = Limit of detection.


\hfill\break

![](../results/figures/figure_3.jpg)

**Figure 3. Histopathologic score and toxin activity varied across donor groups.** (A) Fecal toxin activity detected in some mice post *C. difficile challenge in both moribund and non-moribund mice. (B) Histopathologic summary score for mice at the end of the experiment. Samples were collected for histopathologic scoring on day 10 post-challenge for non-moribund mice or the day the mouse succumbed to the infection for the moribund group (day 2-3 post-challenge). Each point is an individual mouse. Mice are grouped by donor and labeled by the donor letter. Points are colored by donor group. Missing points were from mice with insufficient sample collected for assaying fecal toxin.


\hfill\break

![](../results/figures/figure_4.jpg)

**Figure 4. Individual fecal bacterial community members of the murine gut associate with *C .difficile* infection outcomes.** (A and B) Relative abundance of genera at the time of *C. difficile* challenge (Day 0) that vary significantly by the moribundity and histopathologic summary score or toxin activity by LEfSe analysis. Median (points) and interquartile range (lines) are plotted. Genera are ordered alphabetically to ease comparisons across analyses. (A) Comparison across infection outcome of moribund (colored black) or non-moribund with either a high histopathologic score (score greater than the median score of 5, colored green) or a low histopathologic summary score (score less than the median score of 5, colored light green). (B) Comparison between mice which toxin activity was detected (Toxigenic, colored dark purple) and mice which no toxin activity was detected (Non-toxigenic, colored light purple). (C) Endpoint bacterial OTUs correlate with histopathologic summary score. Each individual mouse is plotted (transparent gray point).  Spearman correlatations are statistically significant after Benjamini-Hochberg correction for multiple comparisons. * indicates bacterial group was unclassified at lower classification ranks.


\hfill\break

![](../results/figures/figure_5.jpg)

**Figure 5. Fecal bacterial community members of the murine gut at the time of *C. difficile* (Day 0) predict outcomes of the infection.** Day 0 bacterial community members grouped by different classification rank were modeled with random forest to predict the infection outcome. The models used the highest taxonomic classification rank that performed as well as the lower ranks. Median (solid points) and interquartile range (lines) of the group relative abundance and each individual (transparent points) relative abundances are plotted. Bacterial groups are ordered by their importance to the model; top taxonomic group has the greatest decrease in performance which its relative abundances are permutted. * indicates bacterial group was unclassified at lower classification ranks. (A) Bacterial members grouped by phyla predicted which mice would have toxin activity detected at any point thoughout the infection (Toxigenic, dark purple). (B) Bacterial members grouped by class predicted which mice would become moribund (dark blue). (C) Bacterial members grouped by genera predicted if the mice would have a high (score greater than the median score of 5, colored dark green) or low (score less than the median score of 5, colored light green) histopathologic summary score.


\hfill\break

![](../results/figures/figure_6.jpg)

**Figure 6. *C. difficile* infection outcome were similar within human fecal donor groups across different *C. difficile* isolates.** Mice colonized with human fecal donor communities G, F, or M were challenged with the isolate from the previous experiments in this study, *C. difficile* ribotype 027 human isolate 431 (blue points), or a different *C. difficile* isolate (purple points). *C. difficile* CFUs were enumerated at the end of the experiment (day 10 post-challenge) and plotted grouped by donor source (top plot). Histopathologic scores from the tissue collected at the endpoint (either day 10 post-challenge or day mice succumbed to infection) were plotted grouped by donor source (bottom plot). Mice that received fecal communities from human donor M succumbed to the infection prior to day 10 post-challenge so were not plated for CFU (DECD = deceased). RT = ribotype, Isolate = human isolate id, LOD = limit of detection.


\hfill\break

![](../results/figures/figure_S1.jpg)

**Figure S1. Histopathologic score of tissue damage at the endpoint of the infection.** Tissue collected at the endpoint, either day 10 post-challenge (Non-moribund) or day mice succumbed to infection (Moribund), were scored from histopathologic damage. Mice (points) are grouped and colored by their human fecal communtity donor. Missing points were from mice with insufficient sample for histopathologic scoring.

\hfill\break

![](../results/figures/figure_S2.jpg)

**Figure S2. Random forest models predict outcomes of the *C. difficile* challenge.** (A-C) Taxonomic classification rank model performance. Relative abundance at the time of *C. difficile* challenge (Day 0) of the bacterial community members grouped by different classification rank were modeled with random forest to predict the infection outcome. The models used the highest taxonomic classification rank that performed as well as the lower ranks. Black rectagle highlights classification rank used to model each outome. (D-F) Model feature importance. Bacterial groups are ordered by their deacrease in area under receiver-operator curve (AUC) when its relative abundances are permutted. Individual relative abundances were added to F since differences are in AUC outside the interquartile range. * indicates bacterial group was unclassified at lower classification ranks. For all plots, median (solid points) and interquartile range (lines) are plotted. (A) Toxin production modeled which mice would have toxin detected during the experiment. (B) Moribundity modeled which mice would succumb to the infection prior to day 10 post-challenge. (C) Histopathologic score modeled which mice would have a high (score greater than the median score of 5) or low (score less than the median score of 5) histopathologic summary score. (D) Bacterial phyla which affect the performance of predicting toxin activity when permutted. (E) Bacterial classes which affect the performance of predicting moribundity when permutted. (D) Bacterial genera which affect the performance of predicting histopathologic score when permutted.

\hfill\break

![](../results/figures/figure_S3.jpg)

**Figure S3. Selection of communities to test for conserved *C. difficile* infection outcome with different *C. difficile* isolate.** (A) Relative abundance of genera in mice from previous experiments which received human fecal donor communities F, G, or M. Each column is an individual mouse. (B) Features from the random forest model that predicted moribundity using relative abundance of bacteria grouped by their genus classification rank and ordered by their decrease in AUC when each feature is permutted. Median (solid points) and interquartile range (lines) change in AUC are plotted. (C) Relative abundance of the most important genera predicting moribundity. Median (solid points) and interquartile range (lines) of the group relative abundance and each individual (transparent points) relative abundances are plotted. * indicates bacterial group was unclassified at lower classification ranks.

