---
title: "initial_analysis_notebook"
author: "Nick Lesniak"
date: "4/13/2021"
output: rmarkdown::github_document
---

```{r setup, echo=FALSE}
knitr::opts_chunk$set(echo = FALSE)
knitr::opts_chunk$set(message = FALSE)
knitr::opts_chunk$set(warning = FALSE)
#knitr::opts_knit$set(root.dir = '../../..')
```

### Process sequencing data

- removed unused fastq files
 
### run fastqs through mothur
    
```{r read in data}
library(tidyverse)

# calculate error rate
error_df <- read.table('../../../data/mothur/sample.error.count', header = T)
total_seqs <- sum(error_df$Sequences)
total_errors <- sum(error_df$Errors * error_df$Sequences)
error_rate <- total_errors / (total_seqs * 253) * 100

read_tsv('../../../data/mothur/sample.final.shared',
         col_types = cols(.default = col_double(),
                          Group = col_character())) %>% 
  select(-numOtus, -label) %>% 
  pivot_longer(cols = -Group, values_to = 'count', names_to = 'otu') %>% 
  group_by(Group) %>% 
  summarise(counts = sum(count)) %>% 
  filter(counts < 2020) %>% 
  arrange(counts) %>% 
  mutate(list = paste0(Group, ' (', counts, ' seqs)')) %>% 
  pull(list) %>% 
  paste(., collapse = ', ')
```

subsampled to 2017, eliminating the 25 samples listed above with their counts
error rate for the data was `r round(error_rate, 2)`%


```{r}
# 
source('../../../code/Sum_OTU_by_Tax.R')

# load in data
metadata <- read_tsv('../../../data/process/metadata_tidy.tsv',
                     col_type = 'cccccDddddDcdddddcdl') 
toxin <- read_tsv('../../../data/process/toxin_tidy.tsv',
                  col_type = 'cccccdc')
histology <- read_tsv('../../../data/process/histology_tidy.tsv',
                      col_type = 'ccccdddcdcc')
human_source <- read_tsv('../../../data/process/human_source_tidy.tsv',
                         col_type = cols(.default = col_character(), age = col_double()))
shared <- read_tsv('../../../data/mothur/sample.final.0.03.subsample.shared',
         col_types = cols(.default = col_double(),
                          Group = col_character())) %>% 
  mutate(Group = ifelse(grepl('_D|DA', Group), Group,
      gsub('D', '_D', Group))) %>% 
  select(-label, -numOtus) %>% 
  pivot_longer(-Group, names_to = 'otu', values_to = 'counts') %>% 
  filter(counts > 0) %>% 
  mutate(relative_abundance = counts/2107 * 100)
taxonomy <- read_tsv('../../../data/process/final.taxonomy.tidy.tsv',
                     col_types = cols(.default = col_character()))
alpha_df <- read_tsv('../../../data/mothur/sample.final.groups.ave-std.summary',
         col_types ='dccdddddd') %>% 
  filter(method == 'ave') %>% 
  mutate(group = ifelse(grepl('_D|DA', group), group,
                        gsub('D', '_D', group)))
nmds_df <- read_tsv('../../../data/mothur/sample.final.thetayc.0.03.lt.ave.nmds.axes',
         col_types ='cdd') %>% 
    mutate(group = ifelse(grepl('_D|DA', group), group,
                        gsub('D', '_D', group)))
source('../../../code/read_dist.R')
beta_df <- read_dist('../../../data/mothur/sample.final.thetayc.0.03.lt.ave.dist') %>% 
      mutate(rows = ifelse(grepl('_D|DA', rows), rows,
                        gsub('D', '_D', rows)),
             columns = ifelse(grepl('_D|DA', columns), columns,
                        gsub('D', '_D', columns)))
donor_colors <- c("#41bbc5", "#7e2640", "#9ad859", "#682dbd", "#728e24", "#e26df8", "#23980d", "#fd048f", "#46e33e", "#fa1bfc", "#155126", "#f9b2d1", "#154975", "#e2c627", "#6294ce")
```

# Figure 1 - Human communities reporducibly colonize mice

Sampled spread of diverse donors for inoculating germ-free mice

```{r fig.width=7, fig.height=5}

human_source_list <- shared %>% 
  filter(grepl('DA', Group)) %>% 
  pull(Group) %>% unique

#nmds_df %>% 
#  filter(group %in% human_source_list) %>% 
#  ggplot(aes(x = axis1, y = axis2, color = outcome)) + 
#    geom_point() + 
#    labs(x = 'NMDS axis 1', y = 'NMDS axis 2', color = NULL) + 
#    theme_bw()
# theta yc shows comparison better
```
  
  
  
Unique communities without conserved structure
```{r, fig.height= 6, fig.width=6}
donor_plot <- shared %>% 
  filter(grepl('DA', Group)) %>% 
  left_join(taxonomy, by = c('otu' = 'OTU')) %>% 
  rename(taxa_level = Class) %>% 
  group_by(Group, taxa_level) %>% 
  summarise(relative_abundance = log10(sum(relative_abundance) + 0.0001)) %>% 
  group_by(taxa_level) %>% 
  mutate(median_ra = median(relative_abundance)) %>% 
  ggplot(aes(x = Group, 
             y = reorder(taxa_level, median_ra), 
             fill = relative_abundance)) + 
    geom_tile() + 
    scale_fill_gradient2(low="white", mid='#0B775E', high = 'black',
		  	limits = c(-2,2), na.value = NA, midpoint = .3,
			  breaks = c(-2.5, -1, 0, 1, 2), labels = c('', '0.1', '1', '10', '100')) + 
	  theme_bw() + 
    labs(x = NULL, y = NULL, fill = NULL) + 
    scale_x_discrete(guide = guide_axis(angle = 45)) + 
    facet_grid(.~Group, scales = 'free_x') + 
    theme(axis.ticks.x = element_blank(),
          axis.text.x = element_blank(),
          legend.position = 'none')

day_0_plot <- metadata %>% 
  filter(day == 0,
         cdiff_strain == 431) %>% 
  select(group, human_source, mouse_id) %>% 
  left_join(shared, by = c('group' = 'Group')) %>% 
  left_join(taxonomy, by = c('otu' = 'OTU')) %>% 
  rename(taxa_level = Class) %>% 
  group_by(group, taxa_level, human_source) %>% 
  summarise(relative_abundance = log10(sum(relative_abundance) + 0.0001)) %>% 
  group_by(taxa_level) %>% 
  mutate(median_ra = median(relative_abundance)) %>% 
  filter(!is.na(taxa_level)) %>% 
  ggplot(aes(x = group, 
             y = reorder(taxa_level, median_ra), 
             fill = relative_abundance)) + 
    geom_tile() + 
    scale_fill_gradient2(low="white", mid='#0B775E', high = 'black',
		  	limits = c(-2,2), na.value = NA, midpoint = .3,
			  breaks = c(-2.5, -1, 0, 1, 2), labels = c('', '0.1', '1', '10', '100')) + 
	  theme_bw() + 
    labs(x = NULL, y = NULL, fill = NULL) + 
    scale_x_discrete(guide = guide_axis(angle = 45)) + 
    facet_grid(.~human_source, scales = 'free_x') + 
    theme(axis.ticks.x = element_blank(),
          axis.text.x = element_blank(),
          legend.position = 'bottom')


cowplot::plot_grid(
  cowplot::plot_grid(NULL, donor_plot, nrow = 1, rel_widths = c(.95,13)),
  day_0_plot, ncol = 1, rel_heights = c(8,9))
```

```{r}
alpha_df <- alpha_df %>% 
  select(group, sobs, invsimpson) %>% 
  left_join(bind_rows(metadata, 
                      tibble(group = unique(metadata$human_source),
                             human_source = unique(metadata$human_source),
                             sample_type = 'donor')),
            by = c('group')) %>% 
  mutate(sample = case_when(sample_type == 'donor' ~ 'Donor',
                            day %in% c(0:10) ~ paste('Day', day),
                            day == mouse_endpoint ~ 'Endpoint',
                            T ~ 'NA'),
         sample = factor(sample, levels = c('Donor', 'Day 0', 'Day 1', 'Day 2', 'Day 3', 
                                            'Day 4', 'Day 5', 'Day 6', 'Day 7', 'Day 8', 'Day 9', 'Day 10'))) %>% 
  filter(sample != 'NA')

alpha_df %>% 
  ggplot(aes(x = sample, y = invsimpson, color = human_source)) + 
    geom_point(alpha = 0.3) + 
    theme_bw() + 
    facet_wrap(human_source~.) + 
    guides(color = 'none')

alpha_df %>% 
  ggplot(aes(x = sample, y = sobs, color = human_source)) + 
    geom_point(alpha = 0.3) + 
    theme_bw() + 
    facet_wrap(human_source~.) + 
    guides(color = 'none')


nmds_df %>% 
  left_join(filter(select(metadata, group, human_source, day), day %in% c(0, 10)), by = c('group')) %>% 
  mutate(human_source = case_when(!is.na(human_source) ~ human_source,
                                  grepl('DA', group) ~ group,
                                  grepl('CON3', group) ~ 'DA00430',
                                  T ~ 'Unknown'),
         sample_type = ifelse(grepl('DA', group), 'human', 'murine')) %>% 
  filter(sample_type == 'human' | day == 0) %>% 
  ggplot(aes(x = axis1, y = axis2, color = human_source)) +
    geom_point() + 
    scale_color_manual(values = donor_colors) + 
    facet_grid(~sample_type) + 
    theme_bw() + 
    guides(color = 'none') + 
    labs(x = 'NMDS axis 1', y = 'NMDS axis 2')
```

```{r}
day_0_samples <- metadata %>% 
  filter(day == 0) %>% 
  select(group, human_source)
day_10_samples <- metadata %>% 
  filter(day == 10) %>% 
  select(group, human_source)


# compare distance between:
#   donor and recipient
#   co-recipients
#   other cages
#   other donors

donor_v_donor <- beta_df %>% 
  filter(grepl('DA', rows),
         grepl('DA', columns)) %>% 
  mutate(comparison = 'Donor\nvs\nDonor')

donor_v_recipent <- bind_rows(
    day_0_samples %>% 
      inner_join(beta_df, by = c('group' = 'rows', 'human_source' = 'columns')),
    day_0_samples %>% 
      inner_join(beta_df, by = c('group' = 'columns', 'human_source' = 'rows'))) %>% 
  mutate(comparison = 'Donor\nvs\nRecipient')

donor_v_recipent_end <- bind_rows(
    day_10_samples %>% 
      inner_join(beta_df, by = c('group' = 'rows', 'human_source' = 'columns')),
    day_10_samples %>% 
      inner_join(beta_df, by = c('group' = 'columns', 'human_source' = 'rows'))) %>% 
  mutate(comparison = 'Donor\nvs\nRecipient End')


corecipient <- day_0_samples %>% 
  left_join(day_0_samples, by = c('human_source')) %>% 
  filter(group.x != group.y) %>% 
  inner_join(beta_df, by = c('group.x' = 'rows', 'group.y' = 'columns')) %>% 
  mutate(comparison = 'Recipients\ncompared to\nothers w/same donor')

other_donors <- beta_df %>% 
  filter((grepl('DA', rows) & grepl('D0', columns)) | 
         (grepl('D0', rows) & grepl('DA', columns))) %>% 
  anti_join(donor_v_recipent, by = c('rows' = 'group', 'columns' = 'human_source')) %>% 
  anti_join(donor_v_recipent, by = c('rows' = 'human_source', 'columns' = 'group')) %>% 
  mutate(comparison = 'Donor\nvs\nNon-recipients')

other_recipients <- beta_df %>% 
  filter(grepl('D0', rows) & grepl('D0', columns)) %>% 
  anti_join(corecipient, by = c('rows' = 'group.x', 'columns' = 'group.y')) %>% 
  mutate(comparison = 'Recipients\ncompared to\nothers w/different donor')

day0_v_day10 <- beta_df %>% 
  separate(rows, c('cageA', 'mouseA', 'dayA')) %>% 
  separate(columns, c('cageB', 'mouseB', 'dayB')) %>% 
  filter(mouseA == mouseB & cageA == cageB) %>% 
  filter((dayA == 'D0' & dayB == 'D10') |
         (dayA == 'D10' & dayB == 'D0')) %>% 
  mutate(comparison = 'Day 0\nvs\nDay 10')



bind_rows(donor_v_donor, donor_v_recipent, corecipient, other_donors, 
          other_recipients, day0_v_day10, donor_v_recipent_end) %>% 
  mutate(comparison = factor(comparison, levels = 
                              c('Donor\nvs\nDonor',
                                'Donor\nvs\nRecipient', 
                                'Recipients\ncompared to\nothers w/same donor', 
                                'Day 0\nvs\nDay 10',
                                'Donor\nvs\nRecipient End',
                                'Donor\nvs\nNon-recipients', 
                                'Recipients\ncompared to\nothers w/different donor'))) %>% 
  ggplot(aes(x = comparison, y = distances)) + 
    stat_summary(fun.data = 'median_hilow', fun.args = (conf.int=0.5)) +
    geom_jitter(alpha = 0.1, width = 0.3) + 
    labs(x = NULL, y = 'Theta YC') +
    theme_bw()

# how do donors distances look by donor?
#donor_v_donor %>% 
#  pivot_longer(cols = c(rows, columns)) %>% 
#  ggplot(aes(x = value, y = distances, color = value)) + geom_point()
#    geom_jitter(width = 0.25) + 
#    scale_color_manual(values = donor_colors) %>% 
#    theme_bw()

corecipient %>% 
  separate(group.x, c('cageA', 'mouseA', 'dayA')) %>% 
  separate(group.y, c('cageB', 'mouseB', 'dayB')) %>% 
  mutate(same_cage = cageA == cageB) %>% 
  ggplot(aes(x = human_source, y = distances, color = human_source, shape = same_cage)) +
    geom_point(position = position_jitterdodge(), width = 0.2) + 
    scale_color_manual(values = donor_colors) + 
    scale_shape_manual(values = c(18,16)) + 
    scale_x_discrete(guide = guide_axis(angle = 45)) + 
    theme_bw() +
    theme(legend.position = 'bottom') + 
    labs(y = 'Theta YC', x = 'Donor Source', color = NULL, shape = 'Same Cage') + 
    guides(color = 'none')
```


# Figure 2 - Mice were colonized without any perturbation

```{r}
metadata %>% 
  filter(cdiff_strain == 431) %>% 
  mutate(human_source = factor(human_source, levels = 
                                c('DA01324', 'DA00953', 'DA00581', 'DA01134', 'DA10148', 
                                  'DA00369', 'DA00430', 'DA10093', 'DA10027', 'DA10034',
                                  'DA10082', 'DA00884', 'DA00578', 'DA00431', 'DA01245'))) %>% 
  ggplot(aes(x = day, y = log10(cdiff_cfu + 60), color = human_source)) +
    stat_summary(fun = 'median', geom = 'line') + 
    geom_point(alpha = 0.3) +
    scale_color_manual(values = donor_colors) + 
    guides(color = 'none') + 
    theme_bw() + 
    labs(x = 'Day', y = 'C. difficile CFU (log10)')
```
```{r}
metadata %>% 
  filter(cdiff_strain == 431,
         day %in% c(1,10)) %>% 
  mutate(human_source = factor(human_source, levels = 
                                c('DA01324', 'DA00953', 'DA00581', 'DA01134', 'DA10148', 
                                  'DA00369', 'DA00430', 'DA10093', 'DA10027', 'DA10034',
                                  'DA10082', 'DA00884', 'DA00578', 'DA00431', 'DA01245'))) %>% 
  ggplot(aes(x = human_source, y = log10(cdiff_cfu + 60), color = human_source)) +
    geom_jitter(width = 0.2) + 
    facet_grid(day~.)+ 
    scale_x_discrete(guide = guide_axis(angle = 45)) + 
    scale_color_manual(values = donor_colors) + 
    theme_bw() + 
    labs(x = 'Donor Source', y = 'C. difficile CFU (log10)') + 
    guides(color = 'none')
```
  
DA00581 is made up of 4 cages, and at day 1 we see 1 cage at ~10^7.5, 2 cages just above 10^5 and 1 cage at 0.
For the cage at 0 there are three mice, one remains at 0, the other two have transient low appearance of cdiff  as seen below
 

```{r}
metadata %>% filter(human_source == 'DA00581') %>% 
  ggplot(aes(x = day, y = log10(cdiff_cfu + 60), shape = mouse_id, color = cage_id)) +
    geom_line() + 
    geom_point() + 
    scale_shape_manual(values = 1:11) + 
    theme_bw() + 
    guides(shape = 'none') + 
    labs(x = 'Day', y = 'C. difficile CFU (log10)')
  
```
  
Points are added to allow identification of individuals. Here we see one mouse has low cfu on days 6 and 7, another mouse on day 10.



# Figure supplemental - weight loss

```{r}
metadata %>% 
  filter(cdiff_strain == 431) %>% 
  mutate(human_source = factor(human_source, levels = 
                                c('DA01324', 'DA00953', 'DA00581', 'DA01134', 'DA10148', 
                                  'DA00369', 'DA00430', 'DA10093', 'DA10027', 'DA10034',
                                  'DA10082', 'DA00884', 'DA00578', 'DA00431', 'DA01245'))) %>% 
  ggplot(aes(x = day, y = percent_weightLoss, color = human_source)) +
    stat_summary(fun = 'median', geom = 'line') + 
    geom_point(alpha = 0.3) +
    scale_color_manual(values = donor_colors) + 
    guides(color = 'none') + 
    theme_bw() + 
    labs(x = 'Day', y = 'Weight (Relative to initial)')
```

```{r}
metadata %>% 
  filter(cdiff_strain == 431,
         day %in% c(2,10)) %>% 
  mutate(human_source = factor(human_source, levels = 
                                c('DA01324', 'DA00953', 'DA00581', 'DA01134', 'DA10148', 
                                  'DA00369', 'DA00430', 'DA10093', 'DA10027', 'DA10034',
                                  'DA10082', 'DA00884', 'DA00578', 'DA00431', 'DA01245'))) %>% 
  ggplot(aes(x = human_source, y = percent_weightLoss, color = human_source)) +
    geom_jitter(width = 0.2) + 
    facet_grid(day~.)+ 
    scale_x_discrete(guide = guide_axis(angle = 45)) + 
    scale_color_manual(values = donor_colors) + 
    theme_bw() + 
    labs(x = 'Donor Source', y = 'Weight (Relative to initial)') + 
    guides(color = 'none')
```


```{r}
#metadata %>% 
#  filter(day == 0 | day == mouse_endpoint) %>% 
#  mutate(time_point = ifelse(day == 0, 'Initial', 'Endpoint'),
#         time_point = factor(time_point, levels = c('Initial', 'Endpoint'))) %>% 
#  left_join(nmds_df, by = c('group')) %>% 
#  ggplot(aes(x = axis1, y = axis2, color = early_euth)) +
#    geom_point() + 
#    facet_grid(.~time_point) +
#    #scale_color_manual(values  = donor_colors) + 
#    theme_bw() + 
#    labs(x = 'NMDS axis 1', y = 'NMDS axis 2', color = 'Moribund') + 
#    theme(legend.position = 'bottom')
#
# not sure if this adds to this figure

```

> NAs on Day 0 for DA01134, DA10034, DA00884  
> Last samples for moribund mice is from cecum  
 

# Figure 3 - Difference in severity

```{r histology}
mouse_by_outcome <- histology %>% 
  mutate(mouse_id = paste0(cage_id, '_', ear_tag)) %>% 
  select(mouse_id, human_source, epithelial_damage, summary_score) %>% 
  left_join(unique(select(metadata, mouse_id, early_euth, cdiff_strain),
            by = c('mouse_id'))) %>% 
  filter(cdiff_strain == 431) %>% 
  group_by(human_source) %>% 
  mutate(severity = mean(epithelial_damage) < 1) %>% 
# using epithelial damage
  mutate(outcome = case_when(early_euth == T ~ 'moribund',
                             severity == T ~ 'mild',
                             severity == F ~ 'severe',
                             T ~ 'NA')) %>% 
# using summary score
#    mutate(outcome_ss = case_when(early_euth == T ~ 'moribund',
#                             summary_score < 4.4 ~ 'mild', # mean = 4.4, median = 5
#                             summary_score > 4.4 ~ 'severe',
#                             T ~ 'NA')) %>% 
  mutate(outcome = factor(outcome, levels = c('mild', 'severe', 'moribund'))) %>% 
  select(mouse_id, outcome)

histology %>% 
  left_join(distinct(select(metadata, mouse_id, early_euth)), by = 'mouse_id') %>% 
  filter(!is.na(early_euth)) %>% 
  mutate(early_euth = ifelse(early_euth, 'Moribund', 'Colonized'),
         mouse_id = paste0(cage_id, '_', ear_tag)) %>% 
  inner_join(mouse_by_outcome, by = c('mouse_id')) %>%
  pivot_longer(cols = c(edema_tissue, inflammation_tissue, epithelial_damage, summary_score),
               names_to = 'histology', values_to = 'score') %>% 
  ggplot(aes(x = early_euth, y = score, , color = early_euth, fill = early_euth)) +
    #stat_summary(fun.data = 'median_hilow', fun.args = (conf.int=0.5), position = position_dodge(width = 0.4)) +
    geom_dotplot(binaxis = 'y', stackdir = 'center', dotsize = 0.5) + 
    #geom_point(position = position_jitterdodge(jitter.width = 0.3), alpha = 0.5) + 
    theme_bw() + 
    labs(x = NULL, y = 'Histology Score', fill = 'Outcome') +
    scale_x_discrete(guide = guide_axis(angle = 45)) + 
    facet_grid(.~histology) + 
    guides(color = 'none') + 
    theme(legend.position = 'bottom')
```


> **Splitting by mean epithelial damage for donor < 1 = 12 mild, 15 severe, 17 moribund**  
> Splitting of median summary score of 5 =  10 mild, 11 severe, 17 moribund  
>        (6 have a summary score of 5)  
> Splitting of mean summary score of 4.4 =  10 mild, 17 severe, 17 moribund  




 What is toxin distribution by outcome?

```{r toxin}
toxin %>% 
  inner_join(select(metadata, group, human_source), by = c('group')) %>% 
  left_join(mouse_by_outcome, by = c('human_source')) %>% 
  filter(!is.na(outcome)) %>% 
  mutate(day = as.numeric(toxin_sample_day),
         outcome = ifelse(outcome == 'moribund', as.character(outcome), 'colonized')) %>% 
  ggplot(aes(x = outcome, y = Log_repiricoal_dilution, fill = outcome, color = outcome)) + 
    stat_summary(fun.data = 'median_hilow', fun.args = (conf.int=0.5), position = position_dodge(width = 0.4)) +
    geom_jitter(width= 0.3, height = 0, alpha = 0.3) + 
    labs(x = 'Day', y = 'Toxin activity') + 
    facet_grid(.~day) + 
    theme_bw() +
    theme(legend.position = 'bottom')
```

## Can we use toxin to split severity?
do mice split into +/- toxin based on outcome of moribund/mild/severe?

```{r}
toxin %>% 
  mutate(mouse_id = ifelse(toxin_sample_type == 'cecalcon', 
                           gsub('C$', '', mouse_id), mouse_id)) %>% 
  group_by(mouse_id) %>% 
  summarise(max_toxin = max(Log_repiricoal_dilution),
            mean_toxin = mean(Log_repiricoal_dilution)) %>% 
  left_join(mouse_by_outcome, by = c('mouse_id')) %>% 
  filter(!is.na(human_source)) %>% 
  pivot_longer(c(max_toxin, mean_toxin)) %>% 
  ggplot(aes(x = outcome, y = value)) + 
    stat_summary(fun.data = 'median_hilow', fun.args = (conf.int=0.5)) +
    geom_jitter(alpha = 0.3, width = 0.3) + 
    facet_wrap(.~name, scales = 'free_y') + 
    labs(x = NULL, y = 'Log reciprocal toxin') + 
    theme_bw()
```


```{r}
## Does toxin explain histological findings?
#toxin %>% 
#  mutate(mouse_id = ifelse(toxin_sample_type == 'cecalcon', 
#                           gsub('C$', '', mouse_id), mouse_id)) %>% 
#  group_by(mouse_id) %>% 
#  mutate(last_sample = max(toxin_sample_day)) %>% 
#  filter(toxin_sample_day == last_sample) %>% 
#  left_join(select(histology, mouse_id, summary_score), by = c('mouse_id')) %>% 
#  left_join(mouse_by_outcome, by = c('mouse_id')) %>% 
#  filter(!is.na(summary_score)) %>% 
#  ggplot(aes(y = Log_repiricoal_dilution, x = summary_score)) + 
#    geom_dotplot(binaxis = 'y', stackdir = 'center', dotsize = 0.5,
#                 aes(fill = as.factor(summary_score))) + 
#    #stat_summary(fun.data = 'median_hilow', fun.args = (conf.int=0.5),
#    #             aes(color = as.factor(summary_score))) +
#    #geom_jitter(width = 0.25, alpha = 0.5) + 
#    scale_color_manual(values = donor_colors) + 
#    scale_fill_manual(values = donor_colors) + 
#    theme_bw() + 
#    scale_x_continuous(breaks = 0:10) + 
#    guides(color = 'none', fill = 'none')
#
#toxin %>% 
#  mutate(mouse_id = ifelse(toxin_sample_type == 'cecalcon', 
#                           gsub('C$', '', mouse_id), mouse_id)) %>% 
#  group_by(mouse_id) %>% 
#  mutate(last_sample = max(toxin_sample_day)) %>% 
#  filter(toxin_sample_day == last_sample) %>% 
#  left_join(select(histology, mouse_id, epithelial_damage), by = c('mouse_id')) %>% 
#  left_join(mouse_by_outcome, by = c('mouse_id')) %>% 
#  filter(!is.na(epithelial_damage)) %>% 
#  ggplot(aes(y = Log_repiricoal_dilution, x = epithelial_damage, 
#             color = as.factor(epithelial_damage))) + 
#    geom_dotplot(binaxis = 'y', stackdir = 'center', dotsize = 0.5,
#                 aes(fill = as.factor(epithelial_damage))) + 
#    #stat_summary(fun.data = 'median_hilow', fun.args = (conf.int=0.5)) +
#    #geom_jitter(width = 0.25, alpha = 0.5) + 
#    scale_color_manual(values = donor_colors) + 
#    scale_fill_manual(values = donor_colors) + 
#    theme_bw() + 
#    scale_x_continuous(breaks = 0:10) + 
#    guides(color = 'none', fill = 'none')
#
# not applicable to our story
```



Mice were challenged with a strain isolated that matched Clostridioides difficile ribotype 027  

> what is published about histology of 027 in C57/B6 mice?   
> how much variation is there in individual isolates of RT027?
> RT027/BI/NAP1, toxinotype III
> VPI 10463 is ribotype 087, toxinotype 0


```{r, individual cfu plots}
#metadata %>% 
#  left_join(mutate(histology, mouse_id = paste0(cage_ID, '_', mouse_ID)) %>% select(-day),
#            by = c('mouse_id')) %>% 
#  mutate(outcome = ifelse(summary_score < 4, 'Low', outcome)) %>% 
#ggplot(aes(x = day, 
#           y = log_cfu, 
#           color = outcome, 
#           group = mouse_id)) + 
#  geom_line() + 
#  facet_wrap(outcome~., ncol = 1) +
#  theme_bw()

shared <- read_tsv('../../../data/mothur/sample.final.0.03.subsample.shared',
         col_types = cols(.default = col_double(),
                          Group = col_character())) %>% 
  mutate(Group = ifelse(grepl('_D|DA', Group), Group,
      gsub('D', '_D', Group))) %>% 
  select(-label, -numOtus) %>% 
  pivot_longer(-Group, names_to = 'otu', values_to = 'counts') %>% 
  mutate(relative_abundance = counts/2107 * 100)


# correlations
# toxin to otu
toxin_otu_corr <- metadata %>% 
  inner_join(toxin, by = 'group') %>% 
  filter(cdiff_strain == 431) %>% 
  select(Group = group, Log_repiricoal_dilution) %>% 
  left_join(shared, by = c('Group')) %>% 
  left_join(taxonomy, by = c('otu' = 'OTU')) %>% 
  filter(!is.na(otu)) %>% 
  group_by(Group, Genus, Log_repiricoal_dilution) %>%
  summarise(counts = sum(counts),
            relative_abundance = sum(relative_abundance)) %>% 
  group_by(Genus) %>% 
  mutate(otu = Genus,
        sd = sd(counts)) %>% 
  filter(sd != 0) %>% 
  nest() %>% 
	mutate(spearman_p = map(.x = data, .f = ~ cor.test(.x$counts, .x$Log_repiricoal_dilution, 
	                                                   method = 'spearman', exact = F)$p.value),
	       spearman_rho = map(.x = data, .f = ~ cor.test(.x$counts, .x$Log_repiricoal_dilution, 
	                                                     method = 'spearman', exact = F)$estimate)) %>% # compare cleared vs colonized
	unnest(c(spearman_p, spearman_rho)) %>% 
	mutate(pvalue = p.adjust(spearman_p, method = 'BH', n = nrow(.))) %>% # correct p values
	filter(pvalue < 0.05) %>% # select only those above 0.05 after pvalue correction
	unnest(data)  

toxin_otu_corr %>% 
  filter(spearman_rho < -0.35 | spearman_rho > 0.35) %>% 
  mutate(relative_abundance = ifelse(relative_abundance == 0, 0.045, relative_abundance)) %>% 
  ggplot(aes(x = relative_abundance, y = Log_repiricoal_dilution, color = otu)) + 
    geom_point(alpha = 0.3) + 
    theme_bw() + 
    facet_wrap(otu~.) + 
    scale_x_log10() + 
    guides(color = 'none') + 
    geom_smooth(method = 'lm', se=F) + 
    labs(x = 'Relative Abundance', y = 'Toxin (log reciprocal dilution)')

```


```{r}
# correlation of histology to otu
plot_hist_corr <- function(i){
  metadata %>% 
    filter(cdiff_strain == 431,
           day == mouse_endpoint) %>% 
    inner_join(histology, by = 'mouse_id') %>% 
    filter(early_euth == F) %>% 
    select(Group = group, test = .data[[i]]) %>% 
    left_join(shared, by = c('Group')) %>% 
    group_by(otu) %>%
    mutate(sd = sd(counts)) %>% 
    filter(!is.na(otu),
           sd != 0) %>% 
    nest() %>% 
  	mutate(spearman_p = map(.x = data, .f = ~ cor.test(.x$counts, .x$test, 
  	                                                   method = 'spearman', exact = F)$p.value),
  	       spearman_rho = map(.x = data, .f = ~ cor.test(.x$counts, .x$test, 
  	                                                     method = 'spearman', exact = F)$estimate)) %>% # compare cleared vs colonized
  	unnest(c(spearman_p, spearman_rho)) %>% 
  	mutate(pvalue = p.adjust(spearman_p, method = 'BH', n = nrow(.))) %>% # correct p values
  	filter(pvalue < 0.05) %>% # select only those above 0.05 after pvalue correction
  	unnest(data) %>% 
    left_join(taxonomy, by = c('otu' = 'OTU')) %>% 
    mutate(tax_otu_label = gsub(' \\(', '\n\\(', tax_otu_label)) %>% 
    ggplot(aes(x = relative_abundance, y = test, color = tax_otu_label)) + 
      geom_point(alpha = 0.3) +
      geom_smooth(method = 'lm', se = F) +
      facet_wrap(tax_otu_label ~ .) + 
      theme_bw() + 
      labs(x = 'Relative Abundance', y = i, title = i) + 
      guides(color = 'none')
}

plot_hist_corr('summary_score')
plot_hist_corr('edema_tissue')
plot_hist_corr('inflammation_tissue')
plot_hist_corr('epithelial_damage')
```
```{r}
# correlation of histology to otu
plot_hist_corr <- function(i){
  metadata %>% 
    filter(cdiff_strain == 431,
           day == mouse_endpoint) %>% 
    inner_join(histology, by = 'mouse_id') %>% 
    filter(early_euth == F) %>% 
    select(Group = group, test = .data[[i]]) %>% 
    left_join(shared, by = c('Group')) %>% 
    group_by(otu) %>%
    mutate(sd = sd(counts)) %>% 
    filter(!is.na(otu),
           sd != 0) %>% 
    nest() %>% 
  	mutate(spearman_p = map(.x = data, .f = ~ cor.test(.x$counts, .x$test, 
  	                                                   method = 'spearman', exact = F)$p.value),
  	       spearman_rho = map(.x = data, .f = ~ cor.test(.x$counts, .x$test, 
  	                                                     method = 'spearman', exact = F)$estimate)) %>% # compare cleared vs colonized
  	unnest(c(spearman_p, spearman_rho)) %>% 
  	mutate(pvalue = p.adjust(spearman_p, method = 'BH', n = nrow(.))) %>% # correct p values
  	filter(pvalue < 0.05) %>% # select only those above 0.05 after pvalue correction
  	unnest(data) %>% 
    left_join(taxonomy, by = c('otu' = 'OTU')) %>% 
    mutate(tax_otu_label = gsub(' \\(', '\n\\(', tax_otu_label)) %>% 
    ggplot(aes(x = relative_abundance, y = test, color = tax_otu_label)) + 
      geom_point(alpha = 0.3) +
      geom_smooth(method = 'lm', se = F) +
      facet_wrap(tax_otu_label ~ .) + 
      theme_bw() + 
      labs(x = 'Relative Abundance', y = i, title = i) + 
      guides(color = 'none')
}

plot_hist_corr('summary_score')
plot_hist_corr('edema_tissue')
plot_hist_corr('inflammation_tissue')
plot_hist_corr('epithelial_damage')
```


```{r}

metadata <- read_tsv('../../../data/process/metadata_tidy.tsv',
		col_type = 'cccccDddddDcdddddcdl') 
shared <- read_tsv('../../../data/mothur/sample.final.0.03.subsample.shared',
		col_types = cols(.default = col_double(),
			Group = col_character())) %>% 
	mutate(Group = ifelse(grepl('_D|DA', Group), Group,
		gsub('D', '_D', Group)))
taxonomy <- read_tsv('../../../data/process/final.taxonomy.tidy.tsv',
                     col_types = cols(.default = col_character()))


plot_lefse_data <- function(file_name){
    lefse_design <- data.table::fread(paste0('../../../data/process/', file_name, '_Genus.design'))
    colnames(lefse_design) <- c('Group', 'class')
  
    lefse_df <- data.table::fread(paste0('../../../data/process/', file_name, '_Genus.0.03.lefse_summary'),
  		fill = T) %>% 
  	filter(!is.na(LDA)) %>% 
  	top_n(20, LDA) %>% 
  	mutate(order = LDA + 10 * as.logical(Class),
  		tax_otu_label = OTU)
  
  lefse_plot <- lefse_df %>% 
  	ggplot(aes(y = LDA, fill = Class, 
  			x = reorder(tax_otu_label, order))) +
  		geom_col() + 
  		theme_bw() + 
  		coord_flip() + 
  		guides(fill = 'none') + 
  		scale_fill_manual(values = c('#8de4d3','#f75b5f')) + 
  		labs(x = NULL, y = 'LDA Score (log 10)')
  
  abundance_plot <- shared %>% 
  	gather(OTU, abundance, matches('Otu\\d*')) %>% 
  	left_join(taxonomy, by = 'OTU') %>% 
  	right_join(lefse_df, by = c('Genus' = 'OTU')) %>% 
    left_join(lefse_design, by = c('Group')) %>% 
  	group_by(Group, Genus, order, class) %>% # removed mouse_id from interactive plot
  	summarise(abundance = sum(abundance)) %>% 
  	mutate(tax_otu_label = Genus) %>% 
  	mutate(abundance = abundance / 21.07,
  		abundance = ifelse(abundance == 0, 0.045, abundance)) %>% 
    #highlight_key(~mouse_id) %>% 
  	ggplot(aes(x = reorder(tax_otu_label, order), 
  			y = abundance, color = class)) + # , shape = mouse_id remove mouse_id from interactive plot
  		stat_summary(fun.data = 'median_hilow', aes(group = class),
  			fun.args = (conf.int=0.5), position = position_dodge(width = 0.5)) +
  		theme_bw() + labs(x = NULL, y = 'Relative Abundance', color = NULL) + 
  		scale_y_log10(breaks=c(0.01, 0.1, 1, 10, 100),
  			labels=c("0","0.1","1","10","100")) +
  		scale_color_manual(values = c('#8de4d3','#f75b5f')) + 
      scale_shape_manual(values = rep(16, 54)) + 
  		geom_hline(yintercept = 0.047, linetype = 'dashed', lwd = 0.3) + 
  		theme(panel.grid.major.y = element_blank(), 
  			panel.grid.minor = element_blank(),
  			legend.position = 'right') + 
  		coord_flip() + 
      guides(shape = 'none')
  
  cowplot::plot_grid(lefse_plot, 
                     abundance_plot + 
                       geom_jitter(position = position_jitterdodge(dodge.width = 0.5), alpha = 0.25) + 
                       theme(axis.ticks.y = element_blank(),
                             axis.text.y = element_blank()), 
                     nrow = 1)
}

#library(plotly)
#gg <- ggplotly(abundance_plot + geom_jitter(position = position_jitterdodge(dodge.width = 0.5)), tooltip = c('mouse_id'))
#highlight(hide_legend(gg), on = "plotly_click", off = "plotly_doubleclick", 
#           color = "black", selectize = T, opacityDim = .2,
#           selected = attrs_selected(showlegend = F, marker = list(symbol = 'X')))

```
## Severe disease
```{r}
plot_lefse_data('severe_disease')
```

## Toxin presence
```{r}
plot_lefse_data('toxin_presence')
```


## Summary score
```{r}
plot_lefse_data('summary_score_hilo')
```


## Epithelial damage
```{r}
plot_lefse_data('epithelial_damage_presence')
```